{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "755d671b",
   "metadata": {},
   "outputs": [],
   "source": [
    "import pathlib\n",
    "import sys\n",
    "import shutil\n",
    "import tempfile\n",
    "import os\n",
    "import subprocess\n",
    "import csv\n",
    "import biom\n",
    "import numpy as np\n",
    "import pandas as pd\n",
    "from qiime2 import Artifact, Metadata\n",
    "from qiime2.plugins import cutadapt \n",
    "from qiime2.plugins.demux.visualizers import summarize\n",
    "from qiime2.plugins.dada2.methods import denoise_single\n",
    "from qiime2.plugins.feature_table.visualizers import summarize as summarize_table"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "e4fbb431",
   "metadata": {},
   "outputs": [],
   "source": [
    "metadata = Metadata.load('../IndexJumping/Sample_Map.txt')\n",
    "demux = Artifact.load('../IndexJumping/demultiplexed_seqs.qza')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "17aedb47",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Running external command line application. This may print messages to stdout and/or stderr.\n",
      "The commands to be run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F01R11_CTAAGGTAACGATGATTCGAGGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F01R11_CTAAGGTAACGATGATTCGAGGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F01R11_CTAAGGTAACGATGATTCGAGGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F01R11_CTAAGGTAACGATGATTCGAGGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.45 s (71 µs/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,241 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads discarded as untrimmed:                9 (0.1%)\n",
      "Reads written (passing filters):         6,241 (99.9%)\n",
      "\n",
      "Total basepairs processed:     1,462,701 bp\n",
      "Total written (filtered):      1,132,725 bp (77.4%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6241 times; 3' trimmed: 6225 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t6\t0.0\t2\t0 0 3 3\n",
      "24\t83\t0.0\t2\t0 20 54 9\n",
      "25\t6120\t0.0\t2\t1440 4239 433 6 2\n",
      "26\t31\t0.0\t2\t0 1 28 2\n",
      "27\t1\t0.0\t2\t0 0 0 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "25\t37\t0.0\t2\t2 29 5 1\n",
      "26\t489\t0.0\t2\t337 103 46 3\n",
      "27\t1479\t0.0\t2\t1084 239 150 5 1\n",
      "28\t4007\t0.0\t2\t1 3889 108 9\n",
      "29\t212\t0.0\t2\t0 0 208 3 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F01R12_CTAAGGTAACGATGAACCACCTA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F01R12_CTAAGGTAACGATGAACCACCTA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F01R12_CTAAGGTAACGATGAACCACCTA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F01R12_CTAAGGTAACGATGAACCACCTA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.38 s (376431 µs/read; 0.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                       1\n",
      "Reads with adapters:                         1 (100.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads discarded as untrimmed:                0 (0.0%)\n",
      "Reads written (passing filters):             1 (100.0%)\n",
      "\n",
      "Total basepairs processed:           233 bp\n",
      "Total written (filtered):            182 bp (78.1%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 1 times; 3' trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "25\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "26\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F01R13_CTAAGGTAACGATGTCCGTTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F01R13_CTAAGGTAACGATGTCCGTTAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F01R13_CTAAGGTAACGATGTCCGTTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F01R13_CTAAGGTAACGATGTCCGTTAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F01R14_CTAAGGTAACGATGACACTCCAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F01R14_CTAAGGTAACGATGACACTCCAA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F01R14_CTAAGGTAACGATGACACTCCAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F01R14_CTAAGGTAACGATGACACTCCAA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F01R15_CTAAGGTAACGATGACCTCTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F01R15_CTAAGGTAACGATGACCTCTAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F01R15_CTAAGGTAACGATGACCTCTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F01R15_CTAAGGTAACGATGACCTCTAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.39 s (392091 µs/read; 0.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                       1\n",
      "Reads with adapters:                         1 (100.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads discarded as untrimmed:                0 (0.0%)\n",
      "Reads written (passing filters):             1 (100.0%)\n",
      "\n",
      "Total basepairs processed:           232 bp\n",
      "Total written (filtered):            181 bp (78.0%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 1 times; 3' trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "25\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "26\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F01R16_CTAAGGTAACGATGTCATCCAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F01R16_CTAAGGTAACGATGTCATCCAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F01R16_CTAAGGTAACGATGTCATCCAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F01R16_CTAAGGTAACGATGTCATCCAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F01R17_CTAAGGTAACGATGACGAATAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F01R17_CTAAGGTAACGATGACGAATAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F01R17_CTAAGGTAACGATGACGAATAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F01R17_CTAAGGTAACGATGACGAATAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F01R18_CTAAGGTAACGATGCAATTGCCT_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F01R18_CTAAGGTAACGATGCAATTGCCT_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F01R18_CTAAGGTAACGATGCAATTGCCT_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F01R18_CTAAGGTAACGATGCAATTGCCT_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F01R19_CTAAGGTAACGATGTCCGACTAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F01R19_CTAAGGTAACGATGTCCGACTAA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F01R19_CTAAGGTAACGATGTCCGACTAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F01R19_CTAAGGTAACGATGTCCGACTAA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F01R20_CTAAGGTAACGATGATGGATCTG_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F01R20_CTAAGGTAACGATGATGGATCTG_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F01R20_CTAAGGTAACGATGATGGATCTG_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F01R20_CTAAGGTAACGATGATGGATCTG_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F02R11_TAAGGAGAACGATGATTCGAGGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F02R11_TAAGGAGAACGATGATTCGAGGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F02R11_TAAGGAGAACGATGATTCGAGGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F02R11_TAAGGAGAACGATGATTCGAGGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.38 s (188957 µs/read; 0.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                       2\n",
      "Reads with adapters:                         2 (100.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   2 (100.0%)\n",
      "Reads discarded as untrimmed:                0 (0.0%)\n",
      "Reads written (passing filters):             0 (0.0%)\n",
      "\n",
      "Total basepairs processed:           103 bp\n",
      "Total written (filtered):              0 bp (0.0%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 2 times; 3' trimmed: 0 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "25\t2\t0.0\t2\t1 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F02R12_TAAGGAGAACGATGAACCACCTA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F02R12_TAAGGAGAACGATGAACCACCTA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F02R12_TAAGGAGAACGATGAACCACCTA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F02R12_TAAGGAGAACGATGAACCACCTA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.41 s (66 µs/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,242 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   8 (0.1%)\n",
      "Reads discarded as untrimmed:                8 (0.1%)\n",
      "Reads written (passing filters):         6,234 (99.7%)\n",
      "\n",
      "Total basepairs processed:     1,453,212 bp\n",
      "Total written (filtered):      1,133,624 bp (78.0%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6242 times; 3' trimmed: 6216 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t6\t0.0\t2\t0 0 6\n",
      "24\t89\t0.0\t2\t0 78 11\n",
      "25\t6134\t0.0\t2\t5469 589 75 1\n",
      "26\t10\t0.0\t2\t0 6 3 1\n",
      "27\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 14\n",
      "25\t229\t0.0\t2\t8 210 11\n",
      "26\t5649\t0.0\t2\t5404 213 31 1\n",
      "27\t291\t0.0\t2\t31 245 14 1\n",
      "28\t33\t0.0\t2\t0 5 28\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F02R13_TAAGGAGAACGATGTCCGTTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F02R13_TAAGGAGAACGATGTCCGTTAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F02R13_TAAGGAGAACGATGTCCGTTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F02R13_TAAGGAGAACGATGTCCGTTAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.42 s (68 µs/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,243 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                  30 (0.5%)\n",
      "Reads discarded as untrimmed:                7 (0.1%)\n",
      "Reads written (passing filters):         6,213 (99.4%)\n",
      "\n",
      "Total basepairs processed:     1,450,020 bp\n",
      "Total written (filtered):      1,126,449 bp (77.7%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6243 times; 3' trimmed: 6231 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "24\t66\t0.0\t2\t0 55 9 2\n",
      "25\t6148\t0.0\t2\t5625 477 45 1\n",
      "26\t28\t0.0\t2\t0 18 10\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t1\t0.0\t2\t1\n",
      "24\t17\t0.0\t2\t0 0 16 1\n",
      "25\t179\t0.0\t2\t7 168 3 1\n",
      "26\t5843\t0.0\t2\t5589 226 28\n",
      "27\t181\t0.0\t2\t16 156 8 1\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F02R14_TAAGGAGAACGATGACACTCCAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F02R14_TAAGGAGAACGATGACACTCCAA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F02R14_TAAGGAGAACGATGACACTCCAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F02R14_TAAGGAGAACGATGACACTCCAA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.41 s (66 µs/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,235 (99.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                  59 (0.9%)\n",
      "Reads discarded as untrimmed:               15 (0.2%)\n",
      "Reads written (passing filters):         6,176 (98.8%)\n",
      "\n",
      "Total basepairs processed:     1,451,741 bp\n",
      "Total written (filtered):      1,121,627 bp (77.3%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6235 times; 3' trimmed: 6209 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t9\t0.0\t2\t0 0 8 1\n",
      "24\t67\t0.0\t2\t0 61 5 1\n",
      "25\t6122\t0.0\t2\t5551 488 83\n",
      "26\t31\t0.0\t2\t0 19 10 1 1\n",
      "27\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t1\t0.0\t2\t0 1\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t108\t0.0\t2\t7 94 7\n",
      "26\t5707\t0.0\t2\t5425 233 46 2 1\n",
      "27\t341\t0.0\t2\t96 232 12 1\n",
      "28\t44\t0.0\t2\t0 7 37\n",
      "29\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F02R15_TAAGGAGAACGATGACCTCTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F02R15_TAAGGAGAACGATGACCTCTAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F02R15_TAAGGAGAACGATGACCTCTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F02R15_TAAGGAGAACGATGACCTCTAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.42 s (67 µs/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,242 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                  19 (0.3%)\n",
      "Reads discarded as untrimmed:                8 (0.1%)\n",
      "Reads written (passing filters):         6,223 (99.6%)\n",
      "\n",
      "Total basepairs processed:     1,451,929 bp\n",
      "Total written (filtered):      1,129,355 bp (77.8%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6242 times; 3' trimmed: 6225 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t6\t0.0\t2\t0 0 6\n",
      "24\t70\t0.0\t2\t0 60 10\n",
      "25\t6143\t0.0\t2\t5706 386 50 1\n",
      "26\t19\t0.0\t2\t0 13 4 2\n",
      "27\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 3 6 1\n",
      "25\t138\t0.0\t2\t17 114 6 1\n",
      "26\t5895\t0.0\t2\t5638 220 36 1\n",
      "27\t164\t0.0\t2\t34 121 9\n",
      "28\t17\t0.0\t2\t0 1 16\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F02R16_TAAGGAGAACGATGTCATCCAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F02R16_TAAGGAGAACGATGTCATCCAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F02R16_TAAGGAGAACGATGTCATCCAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F02R16_TAAGGAGAACGATGTCATCCAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.41 s (66 µs/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,240 (99.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                  17 (0.3%)\n",
      "Reads discarded as untrimmed:               10 (0.2%)\n",
      "Reads written (passing filters):         6,223 (99.6%)\n",
      "\n",
      "Total basepairs processed:     1,451,564 bp\n",
      "Total written (filtered):      1,129,833 bp (77.8%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6240 times; 3' trimmed: 6208 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t10\t0.0\t2\t0 0 10\n",
      "24\t55\t0.0\t2\t0 49 6\n",
      "25\t6162\t0.0\t2\t5649 453 58 2\n",
      "26\t12\t0.0\t2\t0 3 9\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t1\t0.0\t2\t1\n",
      "24\t6\t0.0\t2\t0 0 5 1\n",
      "25\t130\t0.0\t2\t5 118 7\n",
      "26\t5864\t0.0\t2\t5601 224 38 1\n",
      "27\t180\t0.0\t2\t11 161 8\n",
      "28\t26\t0.0\t2\t0 1 25\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F02R17_TAAGGAGAACGATGACGAATAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F02R17_TAAGGAGAACGATGACGAATAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F02R17_TAAGGAGAACGATGACGAATAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F02R17_TAAGGAGAACGATGACGAATAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.36 s (72930 µs/read; 0.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                       5\n",
      "Reads with adapters:                         5 (100.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads discarded as untrimmed:                0 (0.0%)\n",
      "Reads written (passing filters):             5 (100.0%)\n",
      "\n",
      "Total basepairs processed:         1,163 bp\n",
      "Total written (filtered):            908 bp (78.1%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 5 times; 3' trimmed: 5 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "25\t5\t0.0\t2\t5\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "26\t5\t0.0\t2\t5\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F02R18_TAAGGAGAACGATGCAATTGCCT_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F02R18_TAAGGAGAACGATGCAATTGCCT_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F02R18_TAAGGAGAACGATGCAATTGCCT_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F02R18_TAAGGAGAACGATGCAATTGCCT_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.42 s (67 µs/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,238 (99.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                  21 (0.3%)\n",
      "Reads discarded as untrimmed:               12 (0.2%)\n",
      "Reads written (passing filters):         6,217 (99.5%)\n",
      "\n",
      "Total basepairs processed:     1,449,319 bp\n",
      "Total written (filtered):      1,126,004 bp (77.7%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6238 times; 3' trimmed: 6226 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t3\t0.0\t2\t0 0 3\n",
      "24\t55\t0.0\t2\t0 46 8 1\n",
      "25\t6151\t0.0\t2\t5645 449 56 1\n",
      "26\t27\t0.0\t2\t0 15 12\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "21\t1\t0.0\t2\t1\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "24\t10\t0.0\t2\t0 3 7\n",
      "25\t172\t0.0\t2\t13 151 7 1\n",
      "26\t5855\t0.0\t2\t5605 215 35\n",
      "27\t165\t0.0\t2\t15 135 14 1\n",
      "28\t20\t0.0\t2\t0 5 15\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F02R19_TAAGGAGAACGATGTCCGACTAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F02R19_TAAGGAGAACGATGTCCGACTAA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F02R19_TAAGGAGAACGATGTCCGACTAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F02R19_TAAGGAGAACGATGTCCGACTAA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.43 s (69 µs/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,234 (99.7%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   9 (0.1%)\n",
      "Reads discarded as untrimmed:               16 (0.3%)\n",
      "Reads written (passing filters):         6,225 (99.6%)\n",
      "\n",
      "Total basepairs processed:     1,449,962 bp\n",
      "Total written (filtered):      1,127,406 bp (77.8%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6234 times; 3' trimmed: 6221 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t3\t0.0\t2\t0 0 3\n",
      "24\t73\t0.0\t2\t0 65 8\n",
      "25\t6138\t0.0\t2\t5709 392 36 1\n",
      "26\t17\t0.0\t2\t0 7 10\n",
      "27\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "24\t4\t0.0\t2\t0 0 4\n",
      "25\t150\t0.0\t2\t11 133 6\n",
      "26\t5957\t0.0\t2\t5683 244 29 1\n",
      "27\t98\t0.0\t2\t2 87 9\n",
      "28\t10\t0.0\t2\t0 0 10\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F02R20_TAAGGAGAACGATGATGGATCTG_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F02R20_TAAGGAGAACGATGATGGATCTG_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F02R20_TAAGGAGAACGATGATGGATCTG_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F02R20_TAAGGAGAACGATGATGGATCTG_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.43 s (68 µs/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,242 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   5 (0.1%)\n",
      "Reads discarded as untrimmed:                8 (0.1%)\n",
      "Reads written (passing filters):         6,237 (99.8%)\n",
      "\n",
      "Total basepairs processed:     1,452,717 bp\n",
      "Total written (filtered):      1,132,626 bp (78.0%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6242 times; 3' trimmed: 6226 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t8\t0.0\t2\t0 0 6 2\n",
      "24\t138\t0.0\t2\t0 122 14 2\n",
      "25\t6073\t0.0\t2\t5500 500 70 3\n",
      "26\t22\t0.0\t2\t0 15 6 1\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "21\t1\t0.0\t2\t0 1\n",
      "23\t6\t0.0\t2\t0 0 5 1\n",
      "24\t32\t0.0\t2\t0 11 21\n",
      "25\t297\t0.0\t2\t69 214 13 1\n",
      "26\t5709\t0.0\t2\t5394 281 32 2\n",
      "27\t165\t0.0\t2\t22 126 14 3\n",
      "28\t15\t0.0\t2\t0 3 11 1\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F03R11_AAGAGGATTCGATGATTCGAGGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F03R11_AAGAGGATTCGATGATTCGAGGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F03R11_AAGAGGATTCGATGATTCGAGGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F03R11_AAGAGGATTCGATGATTCGAGGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.36 s (360534 µs/read; 0.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                       1\n",
      "Reads with adapters:                         1 (100.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads discarded as untrimmed:                0 (0.0%)\n",
      "Reads written (passing filters):             1 (100.0%)\n",
      "\n",
      "Total basepairs processed:           232 bp\n",
      "Total written (filtered):            181 bp (78.0%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 1 times; 3' trimmed: 1 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "25\t1\t0.0\t2\t1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "26\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F03R12_AAGAGGATTCGATGAACCACCTA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F03R12_AAGAGGATTCGATGAACCACCTA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F03R12_AAGAGGATTCGATGAACCACCTA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F03R12_AAGAGGATTCGATGAACCACCTA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.43 s (68 µs/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,244 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                  43 (0.7%)\n",
      "Reads discarded as untrimmed:                6 (0.1%)\n",
      "Reads written (passing filters):         6,201 (99.2%)\n",
      "\n",
      "Total basepairs processed:     1,451,149 bp\n",
      "Total written (filtered):      1,125,209 bp (77.5%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6244 times; 3' trimmed: 6219 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "24\t87\t0.0\t2\t0 74 12 1\n",
      "25\t6133\t0.0\t2\t5608 478 46 1\n",
      "26\t18\t0.0\t2\t0 10 7 1\n",
      "27\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "22\t2\t0.0\t2\t0 0 2\n",
      "24\t8\t0.0\t2\t0 1 7\n",
      "25\t167\t0.0\t2\t7 153 6 1\n",
      "26\t5780\t0.0\t2\t5536 217 27\n",
      "27\t244\t0.0\t2\t40 191 13\n",
      "28\t18\t0.0\t2\t0 0 16 2\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F03R13_AAGAGGATTCGATGTCCGTTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F03R13_AAGAGGATTCGATGTCCGTTAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F03R13_AAGAGGATTCGATGTCCGTTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F03R13_AAGAGGATTCGATGTCCGTTAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.42 s (67 µs/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,237 (99.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads discarded as untrimmed:               13 (0.2%)\n",
      "Reads written (passing filters):         6,237 (99.8%)\n",
      "\n",
      "Total basepairs processed:     1,452,623 bp\n",
      "Total written (filtered):      1,131,898 bp (77.9%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6237 times; 3' trimmed: 6225 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t6\t0.0\t2\t0 0 6\n",
      "24\t81\t0.0\t2\t0 72 9\n",
      "25\t6130\t0.0\t2\t5686 400 43 1\n",
      "26\t18\t0.0\t2\t0 10 8\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "22\t1\t0.0\t2\t0 1\n",
      "23\t1\t0.0\t2\t0 1\n",
      "24\t10\t0.0\t2\t0 1 9\n",
      "25\t120\t0.0\t2\t13 100 7\n",
      "26\t5921\t0.0\t2\t5659 226 36\n",
      "27\t160\t0.0\t2\t2 153 5\n",
      "28\t12\t0.0\t2\t0 0 11 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F03R14_AAGAGGATTCGATGACACTCCAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F03R14_AAGAGGATTCGATGACACTCCAA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F03R14_AAGAGGATTCGATGACACTCCAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F03R14_AAGAGGATTCGATGACACTCCAA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.41 s (66 µs/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,246 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads discarded as untrimmed:                4 (0.1%)\n",
      "Reads written (passing filters):         6,246 (99.9%)\n",
      "\n",
      "Total basepairs processed:     1,453,308 bp\n",
      "Total written (filtered):      1,134,327 bp (78.1%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6246 times; 3' trimmed: 6229 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t9\t0.0\t2\t0 0 8 1\n",
      "24\t91\t0.0\t2\t0 76 13 1 1\n",
      "25\t6134\t0.0\t2\t5650 422 61 1\n",
      "26\t12\t0.0\t2\t0 8 3 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "24\t11\t0.0\t2\t0 1 10\n",
      "25\t129\t0.0\t2\t10 112 6 0 1\n",
      "26\t5906\t0.0\t2\t5611 267 28\n",
      "27\t152\t0.0\t2\t12 125 12 3\n",
      "28\t30\t0.0\t2\t0 1 28 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F03R15_AAGAGGATTCGATGACCTCTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F03R15_AAGAGGATTCGATGACCTCTAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F03R15_AAGAGGATTCGATGACCTCTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F03R15_AAGAGGATTCGATGACCTCTAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.42 s (68 µs/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,243 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                  14 (0.2%)\n",
      "Reads discarded as untrimmed:                7 (0.1%)\n",
      "Reads written (passing filters):         6,229 (99.7%)\n",
      "\n",
      "Total basepairs processed:     1,453,301 bp\n",
      "Total written (filtered):      1,131,507 bp (77.9%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6243 times; 3' trimmed: 6230 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t5\t0.0\t2\t0 0 4 1\n",
      "24\t99\t0.0\t2\t0 86 12 1\n",
      "25\t6127\t0.0\t2\t5611 446 69 1\n",
      "26\t12\t0.0\t2\t0 5 6 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "22\t1\t0.0\t2\t1\n",
      "23\t1\t0.0\t2\t0 1\n",
      "24\t20\t0.0\t2\t0 0 18 2\n",
      "25\t172\t0.0\t2\t6 158 6 2\n",
      "26\t5815\t0.0\t2\t5550 239 26\n",
      "27\t185\t0.0\t2\t42 134 9\n",
      "28\t36\t0.0\t2\t0 4 32\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F03R16_AAGAGGATTCGATGTCATCCAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F03R16_AAGAGGATTCGATGTCATCCAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F03R16_AAGAGGATTCGATGTCATCCAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F03R16_AAGAGGATTCGATGTCATCCAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.43 s (69 µs/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,237 (99.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   1 (0.0%)\n",
      "Reads discarded as untrimmed:               13 (0.2%)\n",
      "Reads written (passing filters):         6,236 (99.8%)\n",
      "\n",
      "Total basepairs processed:     1,451,917 bp\n",
      "Total written (filtered):      1,131,183 bp (77.9%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6237 times; 3' trimmed: 6220 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t7\t0.0\t2\t0 0 6 1\n",
      "24\t57\t0.0\t2\t0 53 4\n",
      "25\t6151\t0.0\t2\t5669 414 66 2\n",
      "26\t21\t0.0\t2\t0 9 11 1\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1\t0.0\t1\t0 1\n",
      "21\t1\t0.0\t2\t0 1\n",
      "24\t20\t0.0\t2\t0 0 19 1\n",
      "25\t151\t0.0\t2\t3 141 6 1\n",
      "26\t5871\t0.0\t2\t5626 215 30\n",
      "27\t154\t0.0\t2\t25 114 14 1\n",
      "28\t21\t0.0\t2\t0 2 18 1\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F03R17_AAGAGGATTCGATGACGAATAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F03R17_AAGAGGATTCGATGACGAATAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F03R17_AAGAGGATTCGATGACGAATAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F03R17_AAGAGGATTCGATGACGAATAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.42 s (67 µs/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,239 (99.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   3 (0.0%)\n",
      "Reads discarded as untrimmed:               11 (0.2%)\n",
      "Reads written (passing filters):         6,236 (99.8%)\n",
      "\n",
      "Total basepairs processed:     1,454,584 bp\n",
      "Total written (filtered):      1,134,625 bp (78.0%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6239 times; 3' trimmed: 6207 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t9\t0.0\t2\t0 0 7 2\n",
      "24\t99\t0.0\t2\t0 89 9 1\n",
      "25\t6115\t0.0\t2\t5383 630 101 1\n",
      "26\t13\t0.0\t2\t0 6 7\n",
      "27\t3\t0.0\t2\t0 0 2 0 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t21\t0.0\t2\t0 1 20\n",
      "25\t205\t0.0\t2\t21 174 9 1\n",
      "26\t5656\t0.0\t2\t5273 337 46\n",
      "27\t279\t0.0\t2\t58 206 14 1\n",
      "28\t41\t0.0\t2\t0 7 32 1 1\n",
      "29\t5\t0.0\t2\t0 0 4 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F03R18_AAGAGGATTCGATGCAATTGCCT_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F03R18_AAGAGGATTCGATGCAATTGCCT_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F03R18_AAGAGGATTCGATGCAATTGCCT_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F03R18_AAGAGGATTCGATGCAATTGCCT_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.43 s (68 µs/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,240 (99.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   5 (0.1%)\n",
      "Reads discarded as untrimmed:               10 (0.2%)\n",
      "Reads written (passing filters):         6,235 (99.8%)\n",
      "\n",
      "Total basepairs processed:     1,452,060 bp\n",
      "Total written (filtered):      1,131,139 bp (77.9%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6240 times; 3' trimmed: 6216 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t11\t0.0\t2\t0 0 10 0 1\n",
      "24\t71\t0.0\t2\t0 56 13 2\n",
      "25\t6129\t0.0\t2\t5468 606 54 1\n",
      "26\t27\t0.0\t2\t0 12 12 3\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "24\t12\t0.0\t2\t0 1 11\n",
      "25\t225\t0.0\t2\t5 206 12 2\n",
      "26\t5346\t0.0\t2\t5067 233 45 1\n",
      "27\t579\t0.0\t2\t372 196 9 2\n",
      "28\t49\t0.0\t2\t0 38 9 1 1\n",
      "29\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F03R19_AAGAGGATTCGATGTCCGACTAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F03R19_AAGAGGATTCGATGTCCGACTAA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F03R19_AAGAGGATTCGATGTCCGACTAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F03R19_AAGAGGATTCGATGTCCGACTAA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.44 s (70 µs/read; 0.86 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,240 (99.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   8 (0.1%)\n",
      "Reads discarded as untrimmed:               10 (0.2%)\n",
      "Reads written (passing filters):         6,232 (99.7%)\n",
      "\n",
      "Total basepairs processed:     1,453,026 bp\n",
      "Total written (filtered):      1,132,038 bp (77.9%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6240 times; 3' trimmed: 6209 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t4\t0.0\t2\t0 0 4\n",
      "24\t75\t0.0\t2\t0 59 14 2\n",
      "25\t6132\t0.0\t2\t5525 531 75 1\n",
      "26\t28\t0.0\t2\t0 11 16 1\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "22\t2\t0.0\t2\t1 1\n",
      "24\t9\t0.0\t2\t0 1 8\n",
      "25\t187\t0.0\t2\t14 160 11 2\n",
      "26\t5660\t0.0\t2\t5405 220 35\n",
      "27\t311\t0.0\t2\t75 214 21 1\n",
      "28\t40\t0.0\t2\t0 4 35 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F03R20_AAGAGGATTCGATGATGGATCTG_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F03R20_AAGAGGATTCGATGATGGATCTG_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F03R20_AAGAGGATTCGATGATGGATCTG_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F03R20_AAGAGGATTCGATGATGGATCTG_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.42 s (68 µs/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,239 (99.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   1 (0.0%)\n",
      "Reads discarded as untrimmed:               11 (0.2%)\n",
      "Reads written (passing filters):         6,238 (99.8%)\n",
      "\n",
      "Total basepairs processed:     1,452,833 bp\n",
      "Total written (filtered):      1,133,124 bp (78.0%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6239 times; 3' trimmed: 6203 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t19\t0.0\t2\t0 0 16 3\n",
      "24\t176\t0.0\t2\t0 137 35 4\n",
      "25\t6027\t0.0\t2\t5219 679 124 5\n",
      "26\t15\t0.0\t2\t0 6 9\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t5\t0.0\t2\t0 0 4 1\n",
      "24\t39\t0.0\t2\t1 12 25 1\n",
      "25\t357\t0.0\t2\t67 248 40 2\n",
      "26\t5408\t0.0\t2\t4962 381 63 2\n",
      "27\t344\t0.0\t2\t159 160 19 6\n",
      "28\t50\t0.0\t2\t0 15 35\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F04R11_TCGCAATTACGATGATTCGAGGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F04R11_TCGCAATTACGATGATTCGAGGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F04R11_TCGCAATTACGATGATTCGAGGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F04R11_TCGCAATTACGATGATTCGAGGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F04R12_TCGCAATTACGATGAACCACCTA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F04R12_TCGCAATTACGATGAACCACCTA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F04R12_TCGCAATTACGATGAACCACCTA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F04R12_TCGCAATTACGATGAACCACCTA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.42 s (67 µs/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,242 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   7 (0.1%)\n",
      "Reads discarded as untrimmed:                8 (0.1%)\n",
      "Reads written (passing filters):         6,235 (99.8%)\n",
      "\n",
      "Total basepairs processed:     1,456,100 bp\n",
      "Total written (filtered):      1,135,124 bp (78.0%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6242 times; 3' trimmed: 6228 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t7\t0.0\t2\t0 0 6 1\n",
      "24\t88\t0.0\t2\t0 71 16 1\n",
      "25\t6114\t0.0\t2\t5581 487 42 4\n",
      "26\t30\t0.0\t2\t0 21 8 1\n",
      "27\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t1\t0.0\t1\t1\n",
      "24\t11\t0.0\t2\t0 0 9 2\n",
      "25\t152\t0.0\t2\t5 137 8 2\n",
      "26\t5832\t0.0\t2\t5555 242 34 1\n",
      "27\t224\t0.0\t2\t7 199 16 2\n",
      "28\t8\t0.0\t2\t0 0 8\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F04R13_TCGCAATTACGATGTCCGTTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F04R13_TCGCAATTACGATGTCCGTTAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F04R13_TCGCAATTACGATGTCCGTTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F04R13_TCGCAATTACGATGTCCGTTAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.42 s (67 µs/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,239 (99.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                  17 (0.3%)\n",
      "Reads discarded as untrimmed:               11 (0.2%)\n",
      "Reads written (passing filters):         6,222 (99.6%)\n",
      "\n",
      "Total basepairs processed:     1,452,413 bp\n",
      "Total written (filtered):      1,130,380 bp (77.8%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6239 times; 3' trimmed: 6188 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t3\t0.0\t2\t0 0 2 1\n",
      "24\t57\t0.0\t2\t0 42 12 3\n",
      "25\t6148\t0.0\t2\t5585 490 73\n",
      "26\t26\t0.0\t2\t0 19 7\n",
      "27\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1\t0.0\t1\t1\n",
      "21\t1\t0.0\t2\t0 1\n",
      "24\t12\t0.0\t2\t0 0 11 1\n",
      "25\t169\t0.0\t2\t8 153 7 1\n",
      "26\t5765\t0.0\t2\t5504 223 38\n",
      "27\t215\t0.0\t2\t23 165 25 2\n",
      "28\t25\t0.0\t2\t0 7 18\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F04R14_TCGCAATTACGATGACACTCCAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F04R14_TCGCAATTACGATGACACTCCAA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F04R14_TCGCAATTACGATGACACTCCAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F04R14_TCGCAATTACGATGACACTCCAA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.43 s (69 µs/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,242 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                  11 (0.2%)\n",
      "Reads discarded as untrimmed:                8 (0.1%)\n",
      "Reads written (passing filters):         6,231 (99.7%)\n",
      "\n",
      "Total basepairs processed:     1,450,809 bp\n",
      "Total written (filtered):      1,131,979 bp (78.0%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6242 times; 3' trimmed: 6191 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t3\t0.0\t2\t0 0 3\n",
      "24\t59\t0.0\t2\t0 48 8 3\n",
      "25\t6154\t0.0\t2\t5283 782 88 1\n",
      "26\t18\t0.0\t2\t0 13 4 1\n",
      "27\t8\t0.0\t2\t0 0 7 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "11\t1\t0.0\t1\t0 1\n",
      "24\t22\t0.0\t2\t0 1 19 2\n",
      "25\t378\t0.0\t2\t14 349 15\n",
      "26\t5470\t0.0\t2\t5203 232 34 1\n",
      "27\t287\t0.0\t2\t17 255 12 3\n",
      "28\t32\t0.0\t2\t0 3 29\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F04R15_TCGCAATTACGATGACCTCTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F04R15_TCGCAATTACGATGACCTCTAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F04R15_TCGCAATTACGATGACCTCTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F04R15_TCGCAATTACGATGACCTCTAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.46 s (73 µs/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,242 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                  56 (0.9%)\n",
      "Reads discarded as untrimmed:                8 (0.1%)\n",
      "Reads written (passing filters):         6,186 (99.0%)\n",
      "\n",
      "Total basepairs processed:     1,452,062 bp\n",
      "Total written (filtered):      1,124,677 bp (77.5%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6242 times; 3' trimmed: 6214 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t8\t0.0\t2\t0 0 6 1 1\n",
      "24\t63\t0.0\t2\t0 46 16 1\n",
      "25\t6147\t0.0\t2\t5378 693 74 2\n",
      "26\t22\t0.0\t2\t0 15 6 1\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 3 10\n",
      "25\t257\t0.0\t2\t10 238 7 1 1\n",
      "26\t5544\t0.0\t2\t5266 237 39 2\n",
      "27\t351\t0.0\t2\t76 260 14 1\n",
      "28\t46\t0.0\t2\t0 15 30 1\n",
      "29\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F04R16_TCGCAATTACGATGTCATCCAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F04R16_TCGCAATTACGATGTCATCCAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F04R16_TCGCAATTACGATGTCATCCAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F04R16_TCGCAATTACGATGTCATCCAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.42 s (67 µs/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,241 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                  37 (0.6%)\n",
      "Reads discarded as untrimmed:                9 (0.1%)\n",
      "Reads written (passing filters):         6,204 (99.3%)\n",
      "\n",
      "Total basepairs processed:     1,459,709 bp\n",
      "Total written (filtered):      1,134,412 bp (77.7%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6241 times; 3' trimmed: 6177 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t3\t0.0\t2\t0 0 2 1\n",
      "24\t59\t0.0\t2\t0 44 12 3\n",
      "25\t6145\t0.0\t2\t4910 928 303 4\n",
      "26\t28\t0.0\t2\t0 16 11 1\n",
      "27\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "21\t1\t0.0\t2\t0 1\n",
      "23\t2\t0.0\t2\t0 1 1\n",
      "24\t11\t0.0\t2\t1 0 9 1\n",
      "25\t130\t0.0\t2\t5 119 6\n",
      "26\t4988\t0.0\t2\t4762 178 48\n",
      "27\t793\t0.0\t2\t78 679 33 3\n",
      "28\t203\t0.0\t2\t0 5 193 5\n",
      "29\t43\t0.0\t2\t0 5 38\n",
      "30\t5\t0.0\t2\t0 0 5\n",
      "32\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F04R17_TCGCAATTACGATGACGAATAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F04R17_TCGCAATTACGATGACGAATAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F04R17_TCGCAATTACGATGACGAATAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F04R17_TCGCAATTACGATGACGAATAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.42 s (67 µs/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,242 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                  44 (0.7%)\n",
      "Reads discarded as untrimmed:                8 (0.1%)\n",
      "Reads written (passing filters):         6,198 (99.2%)\n",
      "\n",
      "Total basepairs processed:     1,456,446 bp\n",
      "Total written (filtered):      1,129,730 bp (77.6%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6242 times; 3' trimmed: 6214 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "24\t73\t0.0\t2\t0 52 17 3 1\n",
      "25\t6134\t0.0\t2\t5152 787 193 2\n",
      "26\t27\t0.0\t2\t0 18 8 1\n",
      "27\t6\t0.0\t2\t0 0 6\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "4\t1\t24.4\t0\t1\n",
      "17\t1\t0.0\t1\t1\n",
      "23\t2\t0.0\t2\t0 2\n",
      "24\t8\t0.0\t2\t0 2 6\n",
      "25\t158\t0.0\t2\t8 142 8\n",
      "26\t5351\t0.0\t2\t5056 234 59 1 1\n",
      "27\t557\t0.0\t2\t59 467 30 1\n",
      "28\t123\t0.0\t2\t0 8 113 2\n",
      "29\t13\t0.0\t2\t0 1 10 2\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F04R18_TCGCAATTACGATGCAATTGCCT_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F04R18_TCGCAATTACGATGCAATTGCCT_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F04R18_TCGCAATTACGATGCAATTGCCT_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F04R18_TCGCAATTACGATGCAATTGCCT_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.42 s (67 µs/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,239 (99.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                  10 (0.2%)\n",
      "Reads discarded as untrimmed:               11 (0.2%)\n",
      "Reads written (passing filters):         6,229 (99.7%)\n",
      "\n",
      "Total basepairs processed:     1,455,055 bp\n",
      "Total written (filtered):      1,133,110 bp (77.9%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6239 times; 3' trimmed: 6216 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "24\t48\t0.0\t2\t0 39 8 1\n",
      "25\t6150\t0.0\t2\t5437 631 80 2\n",
      "26\t37\t0.0\t2\t0 24 13\n",
      "27\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t11\t0.0\t2\t0 1 10\n",
      "25\t189\t0.0\t2\t3 181 5\n",
      "26\t5624\t0.0\t2\t5392 190 41 1\n",
      "27\t353\t0.0\t2\t21 318 14\n",
      "28\t37\t0.0\t2\t0 1 35 1\n",
      "29\t2\t0.0\t2\t0 1 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F04R19_TCGCAATTACGATGTCCGACTAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F04R19_TCGCAATTACGATGTCCGACTAA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F04R19_TCGCAATTACGATGTCCGACTAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F04R19_TCGCAATTACGATGTCCGACTAA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.44 s (70 µs/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,248 (100.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   4 (0.1%)\n",
      "Reads discarded as untrimmed:                2 (0.0%)\n",
      "Reads written (passing filters):         6,244 (99.9%)\n",
      "\n",
      "Total basepairs processed:     1,454,807 bp\n",
      "Total written (filtered):      1,135,495 bp (78.1%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6248 times; 3' trimmed: 6212 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t64\t0.0\t2\t0 51 11 2\n",
      "25\t6151\t0.0\t2\t5314 646 189 2\n",
      "26\t29\t0.0\t2\t0 19 8 1 1\n",
      "27\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "21\t1\t0.0\t2\t1\n",
      "24\t16\t0.0\t2\t0 1 15\n",
      "25\t141\t0.0\t2\t7 131 3\n",
      "26\t5506\t0.0\t2\t5246 211 48 1\n",
      "27\t410\t0.0\t2\t28 360 21 1\n",
      "28\t119\t0.0\t2\t0 3 112 3 1\n",
      "29\t19\t0.0\t2\t0 6 13\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F04R20_TCGCAATTACGATGATGGATCTG_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F04R20_TCGCAATTACGATGATGGATCTG_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F04R20_TCGCAATTACGATGATGGATCTG_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F04R20_TCGCAATTACGATGATGGATCTG_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.41 s (65 µs/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,246 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   9 (0.1%)\n",
      "Reads discarded as untrimmed:                4 (0.1%)\n",
      "Reads written (passing filters):         6,237 (99.8%)\n",
      "\n",
      "Total basepairs processed:     1,454,163 bp\n",
      "Total written (filtered):      1,133,858 bp (78.0%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6246 times; 3' trimmed: 6222 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t4\t0.0\t2\t0 0 3 1\n",
      "24\t71\t0.0\t2\t0 59 12\n",
      "25\t6152\t0.0\t2\t5456 608 85 3\n",
      "26\t18\t0.0\t2\t0 12 6\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "24\t22\t0.0\t2\t0 6 16\n",
      "25\t255\t0.0\t2\t37 206 11 1\n",
      "26\t5660\t0.0\t2\t5371 264 25\n",
      "27\t247\t0.0\t2\t25 199 20 3\n",
      "28\t35\t0.0\t2\t0 4 31\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F05R11_CAGAAGGAACGATGATTCGAGGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F05R11_CAGAAGGAACGATGATTCGAGGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F05R11_CAGAAGGAACGATGATTCGAGGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F05R11_CAGAAGGAACGATGATTCGAGGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F05R12_CAGAAGGAACGATGAACCACCTA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F05R12_CAGAAGGAACGATGAACCACCTA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F05R12_CAGAAGGAACGATGAACCACCTA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F05R12_CAGAAGGAACGATGAACCACCTA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.42 s (67 µs/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,244 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                  28 (0.4%)\n",
      "Reads discarded as untrimmed:                6 (0.1%)\n",
      "Reads written (passing filters):         6,216 (99.5%)\n",
      "\n",
      "Total basepairs processed:     1,449,031 bp\n",
      "Total written (filtered):      1,128,383 bp (77.9%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6244 times; 3' trimmed: 6221 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "24\t63\t0.0\t2\t0 55 7 1\n",
      "25\t6158\t0.0\t2\t5677 434 47\n",
      "26\t17\t0.0\t2\t0 12 5\n",
      "27\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "14\t1\t0.0\t1\t0 1\n",
      "16\t1\t0.0\t1\t1\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "24\t13\t0.0\t2\t0 1 12\n",
      "25\t154\t0.0\t2\t3 145 6\n",
      "26\t5859\t0.0\t2\t5648 186 25\n",
      "27\t179\t0.0\t2\t4 163 11 1\n",
      "28\t13\t0.0\t2\t0 3 10\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F05R13_CAGAAGGAACGATGTCCGTTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F05R13_CAGAAGGAACGATGTCCGTTAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F05R13_CAGAAGGAACGATGTCCGTTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F05R13_CAGAAGGAACGATGTCCGTTAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.42 s (67 µs/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,248 (100.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   2 (0.0%)\n",
      "Reads discarded as untrimmed:                2 (0.0%)\n",
      "Reads written (passing filters):         6,246 (99.9%)\n",
      "\n",
      "Total basepairs processed:     1,452,940 bp\n",
      "Total written (filtered):      1,134,243 bp (78.1%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6248 times; 3' trimmed: 6221 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "24\t41\t0.0\t2\t0 39 2\n",
      "25\t6178\t0.0\t2\t5701 440 36 1\n",
      "26\t25\t0.0\t2\t0 15 10\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t1\t0.0\t1\t0 1\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "24\t6\t0.0\t2\t0 0 6\n",
      "25\t186\t0.0\t2\t4 178 4\n",
      "26\t5840\t0.0\t2\t5643 168 29\n",
      "27\t178\t0.0\t2\t27 146 4 1\n",
      "28\t9\t0.0\t2\t0 1 8\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F05R14_CAGAAGGAACGATGACACTCCAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F05R14_CAGAAGGAACGATGACACTCCAA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F05R14_CAGAAGGAACGATGACACTCCAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F05R14_CAGAAGGAACGATGACACTCCAA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.41 s (66 µs/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,244 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                  34 (0.5%)\n",
      "Reads discarded as untrimmed:                6 (0.1%)\n",
      "Reads written (passing filters):         6,210 (99.4%)\n",
      "\n",
      "Total basepairs processed:     1,446,902 bp\n",
      "Total written (filtered):      1,127,895 bp (78.0%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6244 times; 3' trimmed: 6200 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t8\t0.0\t2\t0 0 7 0 1\n",
      "24\t116\t0.0\t2\t0 93 16 7\n",
      "25\t6109\t0.0\t2\t5264 743 99 3\n",
      "26\t10\t0.0\t2\t0 7 3\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "22\t1\t0.0\t2\t0 1\n",
      "24\t28\t0.0\t2\t0 1 25 2\n",
      "25\t285\t0.0\t2\t19 252 11 3\n",
      "26\t5398\t0.0\t2\t5063 301 31 3\n",
      "27\t407\t0.0\t2\t138 249 18 2\n",
      "28\t78\t0.0\t2\t0 39 39\n",
      "29\t3\t0.0\t2\t0 0 2 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F05R15_CAGAAGGAACGATGACCTCTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F05R15_CAGAAGGAACGATGACCTCTAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F05R15_CAGAAGGAACGATGACCTCTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F05R15_CAGAAGGAACGATGACCTCTAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.42 s (67 µs/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,246 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads discarded as untrimmed:                4 (0.1%)\n",
      "Reads written (passing filters):         6,246 (99.9%)\n",
      "\n",
      "Total basepairs processed:     1,451,321 bp\n",
      "Total written (filtered):      1,132,473 bp (78.0%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6246 times; 3' trimmed: 6225 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t3\t0.0\t2\t0 0 3\n",
      "24\t61\t0.0\t2\t0 55 6\n",
      "25\t6163\t0.0\t2\t5727 398 38\n",
      "26\t18\t0.0\t2\t0 15 3\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "24\t7\t0.0\t2\t0 1 6\n",
      "25\t170\t0.0\t2\t14 150 6\n",
      "26\t5896\t0.0\t2\t5664 212 20\n",
      "27\t139\t0.0\t2\t29 102 8\n",
      "28\t12\t0.0\t2\t0 2 10\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F05R16_CAGAAGGAACGATGTCATCCAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F05R16_CAGAAGGAACGATGTCATCCAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F05R16_CAGAAGGAACGATGTCATCCAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F05R16_CAGAAGGAACGATGTCATCCAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.42 s (67 µs/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,245 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   3 (0.0%)\n",
      "Reads discarded as untrimmed:                5 (0.1%)\n",
      "Reads written (passing filters):         6,242 (99.9%)\n",
      "\n",
      "Total basepairs processed:     1,453,156 bp\n",
      "Total written (filtered):      1,133,603 bp (78.0%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6245 times; 3' trimmed: 6220 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "24\t29\t0.0\t2\t0 27 1 1\n",
      "25\t6194\t0.0\t2\t5626 503 65\n",
      "26\t21\t0.0\t2\t0 12 8 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1\t0.0\t1\t1\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "24\t12\t0.0\t2\t0 1 10 1\n",
      "25\t144\t0.0\t2\t8 134 2\n",
      "26\t5756\t0.0\t2\t5548 191 16 1\n",
      "27\t264\t0.0\t2\t44 210 10\n",
      "28\t39\t0.0\t2\t0 6 33\n",
      "29\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F05R17_CAGAAGGAACGATGACGAATAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F05R17_CAGAAGGAACGATGACGAATAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F05R17_CAGAAGGAACGATGACGAATAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F05R17_CAGAAGGAACGATGACGAATAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.43 s (69 µs/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,240 (99.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                  68 (1.1%)\n",
      "Reads discarded as untrimmed:               10 (0.2%)\n",
      "Reads written (passing filters):         6,172 (98.8%)\n",
      "\n",
      "Total basepairs processed:     1,455,124 bp\n",
      "Total written (filtered):      1,123,956 bp (77.2%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6240 times; 3' trimmed: 6217 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t86\t0.0\t2\t0 68 15 3\n",
      "25\t6130\t0.0\t2\t5444 617 66 3\n",
      "26\t24\t0.0\t2\t0 16 8\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "21\t1\t0.0\t2\t0 1\n",
      "24\t11\t0.0\t2\t0 1 10\n",
      "25\t250\t0.0\t2\t24 213 13\n",
      "26\t5447\t0.0\t2\t5214 202 28 3\n",
      "27\t469\t0.0\t2\t183 275 9 2\n",
      "28\t36\t0.0\t2\t0 9 26 1\n",
      "29\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F05R18_CAGAAGGAACGATGCAATTGCCT_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F05R18_CAGAAGGAACGATGCAATTGCCT_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F05R18_CAGAAGGAACGATGCAATTGCCT_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F05R18_CAGAAGGAACGATGCAATTGCCT_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.40 s (65 µs/read; 0.93 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,249 (100.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                  65 (1.0%)\n",
      "Reads discarded as untrimmed:                1 (0.0%)\n",
      "Reads written (passing filters):         6,184 (98.9%)\n",
      "\n",
      "Total basepairs processed:     1,452,542 bp\n",
      "Total written (filtered):      1,124,383 bp (77.4%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6249 times; 3' trimmed: 6225 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t2\t0.0\t2\t0 0 1 1\n",
      "24\t64\t0.0\t2\t0 48 12 4\n",
      "25\t6152\t0.0\t2\t5323 751 77 1\n",
      "26\t29\t0.0\t2\t0 21 7 1\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 2 11\n",
      "25\t303\t0.0\t2\t11 281 9 2\n",
      "26\t5429\t0.0\t2\t5204 190 35\n",
      "27\t439\t0.0\t2\t84 340 13 2\n",
      "28\t39\t0.0\t2\t0 7 29 3\n",
      "29\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F05R19_CAGAAGGAACGATGTCCGACTAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F05R19_CAGAAGGAACGATGTCCGACTAA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F05R19_CAGAAGGAACGATGTCCGACTAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F05R19_CAGAAGGAACGATGTCCGACTAA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.44 s (71 µs/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,245 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                  19 (0.3%)\n",
      "Reads discarded as untrimmed:                5 (0.1%)\n",
      "Reads written (passing filters):         6,226 (99.6%)\n",
      "\n",
      "Total basepairs processed:     1,448,425 bp\n",
      "Total written (filtered):      1,127,467 bp (77.8%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6245 times; 3' trimmed: 6229 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "24\t62\t0.0\t2\t0 53 8 1\n",
      "25\t6166\t0.0\t2\t5589 510 67\n",
      "26\t12\t0.0\t2\t0 8 2 2\n",
      "27\t4\t0.0\t2\t0 0 3 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 3 17\n",
      "25\t185\t0.0\t2\t18 155 11 1\n",
      "26\t5774\t0.0\t2\t5540 214 20\n",
      "27\t229\t0.0\t2\t16 198 14 1\n",
      "28\t21\t0.0\t2\t0 0 19 2\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F05R20_CAGAAGGAACGATGATGGATCTG_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F05R20_CAGAAGGAACGATGATGGATCTG_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F05R20_CAGAAGGAACGATGATGGATCTG_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F05R20_CAGAAGGAACGATGATGGATCTG_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.42 s (67 µs/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,244 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads discarded as untrimmed:                6 (0.1%)\n",
      "Reads written (passing filters):         6,244 (99.9%)\n",
      "\n",
      "Total basepairs processed:     1,454,116 bp\n",
      "Total written (filtered):      1,135,099 bp (78.1%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6244 times; 3' trimmed: 6221 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t9\t0.0\t2\t0 0 7 2\n",
      "24\t118\t0.0\t2\t0 89 26 2 1\n",
      "25\t6091\t0.0\t2\t5323 676 91 1\n",
      "26\t26\t0.0\t2\t0 17 5 4\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t28\t0.0\t2\t0 5 23\n",
      "25\t375\t0.0\t2\t30 322 23\n",
      "26\t5517\t0.0\t2\t5237 241 34 5\n",
      "27\t260\t0.0\t2\t34 207 16 3\n",
      "28\t40\t0.0\t2\t0 6 32 1 1\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F06R11_CTGCAAGTTCGATGATTCGAGGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F06R11_CTGCAAGTTCGATGATTCGAGGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F06R11_CTGCAAGTTCGATGATTCGAGGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F06R11_CTGCAAGTTCGATGATTCGAGGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F06R12_CTGCAAGTTCGATGAACCACCTA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F06R12_CTGCAAGTTCGATGAACCACCTA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F06R12_CTGCAAGTTCGATGAACCACCTA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F06R12_CTGCAAGTTCGATGAACCACCTA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.41 s (65 µs/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,250 (100.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads discarded as untrimmed:                0 (0.0%)\n",
      "Reads written (passing filters):         6,250 (100.0%)\n",
      "\n",
      "Total basepairs processed:     1,451,396 bp\n",
      "Total written (filtered):      1,133,087 bp (78.1%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6250 times; 3' trimmed: 6237 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t6\t0.0\t2\t0 0 5 1\n",
      "24\t54\t0.0\t2\t0 51 2 1\n",
      "25\t6169\t0.0\t2\t5849 297 23\n",
      "26\t19\t0.0\t2\t0 16 3\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "12\t1\t0.0\t1\t1\n",
      "24\t7\t0.0\t2\t0 0 7\n",
      "25\t128\t0.0\t2\t11 114 3\n",
      "26\t6008\t0.0\t2\t5820 172 15 1\n",
      "27\t90\t0.0\t2\t4 78 7 1\n",
      "28\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F06R13_CTGCAAGTTCGATGTCCGTTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F06R13_CTGCAAGTTCGATGTCCGTTAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F06R13_CTGCAAGTTCGATGTCCGTTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F06R13_CTGCAAGTTCGATGTCCGTTAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.41 s (66 µs/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,241 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                  26 (0.4%)\n",
      "Reads discarded as untrimmed:                9 (0.1%)\n",
      "Reads written (passing filters):         6,215 (99.4%)\n",
      "\n",
      "Total basepairs processed:     1,453,175 bp\n",
      "Total written (filtered):      1,129,690 bp (77.7%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6241 times; 3' trimmed: 6203 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t10\t0.0\t2\t0 0 8 1 1\n",
      "24\t106\t0.0\t2\t0 92 13 1\n",
      "25\t6107\t0.0\t2\t5358 667 81 1\n",
      "26\t17\t0.0\t2\t0 12 2 3\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t20\t0.0\t2\t0 0 18 2\n",
      "25\t306\t0.0\t2\t11 281 12 2\n",
      "26\t5548\t0.0\t2\t5273 241 34\n",
      "27\t295\t0.0\t2\t37 247 11\n",
      "28\t34\t0.0\t2\t0 1 30 2 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F06R14_CTGCAAGTTCGATGACACTCCAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F06R14_CTGCAAGTTCGATGACACTCCAA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F06R14_CTGCAAGTTCGATGACACTCCAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F06R14_CTGCAAGTTCGATGACACTCCAA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.41 s (66 µs/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,248 (100.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   3 (0.0%)\n",
      "Reads discarded as untrimmed:                2 (0.0%)\n",
      "Reads written (passing filters):         6,245 (99.9%)\n",
      "\n",
      "Total basepairs processed:     1,452,203 bp\n",
      "Total written (filtered):      1,133,962 bp (78.1%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6248 times; 3' trimmed: 6222 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t9\t0.0\t2\t0 0 9\n",
      "24\t137\t0.0\t2\t0 125 12\n",
      "25\t6079\t0.0\t2\t5533 494 50 2\n",
      "26\t20\t0.0\t2\t0 14 5 1\n",
      "27\t3\t0.0\t2\t0 0 2 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "15\t1\t0.0\t1\t0 1\n",
      "21\t1\t0.0\t2\t0 1\n",
      "22\t1\t0.0\t2\t0 0 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "24\t13\t0.0\t2\t1 3 9\n",
      "25\t230\t0.0\t2\t13 211 4 2\n",
      "26\t5804\t0.0\t2\t5486 279 38 1\n",
      "27\t155\t0.0\t2\t7 137 10 1\n",
      "28\t15\t0.0\t2\t0 1 14\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F06R15_CTGCAAGTTCGATGACCTCTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F06R15_CTGCAAGTTCGATGACCTCTAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F06R15_CTGCAAGTTCGATGACCTCTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F06R15_CTGCAAGTTCGATGACCTCTAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.43 s (68 µs/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,246 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   1 (0.0%)\n",
      "Reads discarded as untrimmed:                4 (0.1%)\n",
      "Reads written (passing filters):         6,245 (99.9%)\n",
      "\n",
      "Total basepairs processed:     1,451,346 bp\n",
      "Total written (filtered):      1,132,496 bp (78.0%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6246 times; 3' trimmed: 6226 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t5\t0.0\t2\t0 0 5\n",
      "24\t67\t0.0\t2\t0 60 4 3\n",
      "25\t6151\t0.0\t2\t5728 377 46\n",
      "26\t20\t0.0\t2\t0 15 4 1\n",
      "27\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "24\t6\t0.0\t2\t0 1 5\n",
      "25\t179\t0.0\t2\t13 159 5 2\n",
      "26\t5918\t0.0\t2\t5693 202 22 1\n",
      "27\t97\t0.0\t2\t2 90 5\n",
      "28\t25\t0.0\t2\t0 0 24 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F06R16_CTGCAAGTTCGATGTCATCCAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F06R16_CTGCAAGTTCGATGTCATCCAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F06R16_CTGCAAGTTCGATGTCATCCAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F06R16_CTGCAAGTTCGATGTCATCCAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.41 s (66 µs/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,244 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   7 (0.1%)\n",
      "Reads discarded as untrimmed:                6 (0.1%)\n",
      "Reads written (passing filters):         6,237 (99.8%)\n",
      "\n",
      "Total basepairs processed:     1,450,974 bp\n",
      "Total written (filtered):      1,132,043 bp (78.0%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6244 times; 3' trimmed: 6194 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t7\t0.0\t2\t0 0 7\n",
      "24\t83\t0.0\t2\t0 69 13 0 1\n",
      "25\t6135\t0.0\t2\t5575 507 52 1\n",
      "26\t18\t0.0\t2\t0 16 2\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 3 6 1\n",
      "25\t239\t0.0\t2\t2 231 6\n",
      "26\t5759\t0.0\t2\t5522 215 22\n",
      "27\t159\t0.0\t2\t3 141 15\n",
      "28\t27\t0.0\t2\t0 0 26 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F06R17_CTGCAAGTTCGATGACGAATAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F06R17_CTGCAAGTTCGATGACGAATAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F06R17_CTGCAAGTTCGATGACGAATAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F06R17_CTGCAAGTTCGATGACGAATAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.42 s (67 µs/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,238 (99.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                  12 (0.2%)\n",
      "Reads discarded as untrimmed:               12 (0.2%)\n",
      "Reads written (passing filters):         6,226 (99.6%)\n",
      "\n",
      "Total basepairs processed:     1,450,087 bp\n",
      "Total written (filtered):      1,128,377 bp (77.8%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6238 times; 3' trimmed: 6227 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t3\t0.0\t2\t0 0 3\n",
      "24\t129\t0.0\t2\t0 108 20 1\n",
      "25\t6082\t0.0\t2\t5560 476 45 1\n",
      "26\t22\t0.0\t2\t0 16 5 1\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "13\t1\t0.0\t1\t0 1\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "24\t11\t0.0\t2\t0 1 9 1\n",
      "25\t171\t0.0\t2\t8 153 9 1\n",
      "26\t5829\t0.0\t2\t5535 265 29\n",
      "27\t194\t0.0\t2\t8 176 10\n",
      "28\t17\t0.0\t2\t0 1 16\n",
      "29\t1\t0.0\t2\t0 0 0 1\n",
      "30\t1\t0.0\t2\t0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F06R18_CTGCAAGTTCGATGCAATTGCCT_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F06R18_CTGCAAGTTCGATGCAATTGCCT_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F06R18_CTGCAAGTTCGATGCAATTGCCT_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F06R18_CTGCAAGTTCGATGCAATTGCCT_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.43 s (69 µs/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,241 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                  51 (0.8%)\n",
      "Reads discarded as untrimmed:                9 (0.1%)\n",
      "Reads written (passing filters):         6,190 (99.0%)\n",
      "\n",
      "Total basepairs processed:     1,453,573 bp\n",
      "Total written (filtered):      1,125,536 bp (77.4%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6241 times; 3' trimmed: 6223 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t6\t0.0\t2\t0 0 6\n",
      "24\t77\t0.0\t2\t0 68 8 1\n",
      "25\t6135\t0.0\t2\t5456 621 58\n",
      "26\t22\t0.0\t2\t0 16 6\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t10\t0.0\t2\t0 1 9\n",
      "25\t263\t0.0\t2\t15 241 6 1\n",
      "26\t5631\t0.0\t2\t5383 215 33\n",
      "27\t299\t0.0\t2\t41 245 13\n",
      "28\t19\t0.0\t2\t0 3 16\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F06R19_CTGCAAGTTCGATGTCCGACTAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F06R19_CTGCAAGTTCGATGTCCGACTAA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F06R19_CTGCAAGTTCGATGTCCGACTAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F06R19_CTGCAAGTTCGATGTCCGACTAA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.43 s (69 µs/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,245 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                  16 (0.3%)\n",
      "Reads discarded as untrimmed:                5 (0.1%)\n",
      "Reads written (passing filters):         6,229 (99.7%)\n",
      "\n",
      "Total basepairs processed:     1,453,454 bp\n",
      "Total written (filtered):      1,131,961 bp (77.9%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6245 times; 3' trimmed: 6216 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t5\t0.0\t2\t0 0 5\n",
      "24\t81\t0.0\t2\t0 72 8 1\n",
      "25\t6141\t0.0\t2\t5516 547 77 1\n",
      "26\t18\t0.0\t2\t0 15 3\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "24\t17\t0.0\t2\t0 6 11\n",
      "25\t199\t0.0\t2\t36 155 8\n",
      "26\t5694\t0.0\t2\t5428 239 26 1\n",
      "27\t267\t0.0\t2\t24 232 11\n",
      "28\t37\t0.0\t2\t0 1 35 1\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F06R20_CTGCAAGTTCGATGATGGATCTG_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F06R20_CTGCAAGTTCGATGATGGATCTG_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F06R20_CTGCAAGTTCGATGATGGATCTG_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F06R20_CTGCAAGTTCGATGATGGATCTG_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.42 s (68 µs/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,243 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                  29 (0.5%)\n",
      "Reads discarded as untrimmed:                7 (0.1%)\n",
      "Reads written (passing filters):         6,214 (99.4%)\n",
      "\n",
      "Total basepairs processed:     1,454,474 bp\n",
      "Total written (filtered):      1,131,323 bp (77.8%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6243 times; 3' trimmed: 6205 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t12\t0.0\t2\t0 0 12\n",
      "24\t170\t0.0\t2\t0 133 32 4 1\n",
      "25\t6048\t0.0\t2\t5196 703 143 5 1\n",
      "26\t11\t0.0\t2\t0 9 1 1\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "22\t1\t0.0\t2\t1\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "24\t32\t0.0\t2\t0 7 22 3\n",
      "25\t369\t0.0\t2\t49 294 24 2\n",
      "26\t5368\t0.0\t2\t5076 231 59 2\n",
      "27\t362\t0.0\t2\t33 305 22 1 1\n",
      "28\t66\t0.0\t2\t1 6 57 1 1\n",
      "29\t5\t0.0\t2\t0 0 4 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F07R11_TTCGTGATTCGATGATTCGAGGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F07R11_TTCGTGATTCGATGATTCGAGGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F07R11_TTCGTGATTCGATGATTCGAGGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F07R11_TTCGTGATTCGATGATTCGAGGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F07R12_TTCGTGATTCGATGAACCACCTA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F07R12_TTCGTGATTCGATGAACCACCTA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F07R12_TTCGTGATTCGATGAACCACCTA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F07R12_TTCGTGATTCGATGAACCACCTA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.40 s (64 µs/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,245 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   2 (0.0%)\n",
      "Reads discarded as untrimmed:                5 (0.1%)\n",
      "Reads written (passing filters):         6,243 (99.9%)\n",
      "\n",
      "Total basepairs processed:     1,452,116 bp\n",
      "Total written (filtered):      1,132,387 bp (78.0%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6245 times; 3' trimmed: 6231 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t3\t0.0\t2\t0 0 3\n",
      "24\t43\t0.0\t2\t0 37 3 3\n",
      "25\t6165\t0.0\t2\t5653 471 40 1\n",
      "26\t31\t0.0\t2\t0 22 9\n",
      "27\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "23\t2\t0.0\t2\t1 0 1\n",
      "24\t5\t0.0\t2\t0 0 4 1\n",
      "25\t179\t0.0\t2\t8 162 6 3\n",
      "26\t5749\t0.0\t2\t5519 202 28\n",
      "27\t275\t0.0\t2\t113 154 8\n",
      "28\t19\t0.0\t2\t0 10 9\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F07R13_TTCGTGATTCGATGTCCGTTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F07R13_TTCGTGATTCGATGTCCGTTAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F07R13_TTCGTGATTCGATGTCCGTTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F07R13_TTCGTGATTCGATGTCCGTTAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.42 s (67 µs/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,248 (100.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   5 (0.1%)\n",
      "Reads discarded as untrimmed:                2 (0.0%)\n",
      "Reads written (passing filters):         6,243 (99.9%)\n",
      "\n",
      "Total basepairs processed:     1,450,316 bp\n",
      "Total written (filtered):      1,130,849 bp (78.0%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6248 times; 3' trimmed: 6237 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t6\t0.0\t2\t0 0 5 1\n",
      "24\t51\t0.0\t2\t0 43 7 1\n",
      "25\t6177\t0.0\t2\t5599 524 54\n",
      "26\t14\t0.0\t2\t0 8 5 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t16\t0.0\t2\t0 1 15\n",
      "25\t270\t0.0\t2\t8 252 9 1\n",
      "26\t5733\t0.0\t2\t5540 165 28\n",
      "27\t202\t0.0\t2\t40 156 5 1\n",
      "28\t16\t0.0\t2\t0 1 14 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F07R14_TTCGTGATTCGATGACACTCCAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F07R14_TTCGTGATTCGATGACACTCCAA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F07R14_TTCGTGATTCGATGACACTCCAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F07R14_TTCGTGATTCGATGACACTCCAA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.41 s (66 µs/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,249 (100.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   9 (0.1%)\n",
      "Reads discarded as untrimmed:                1 (0.0%)\n",
      "Reads written (passing filters):         6,240 (99.8%)\n",
      "\n",
      "Total basepairs processed:     1,451,082 bp\n",
      "Total written (filtered):      1,131,352 bp (78.0%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6249 times; 3' trimmed: 6228 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t6\t0.0\t2\t0 0 4 2\n",
      "24\t89\t0.0\t2\t0 77 8 3 1\n",
      "25\t6129\t0.0\t2\t5532 532 65\n",
      "26\t23\t0.0\t2\t0 18 5\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t1\t0.0\t2\t0 1\n",
      "24\t17\t0.0\t2\t1 0 15 1\n",
      "25\t224\t0.0\t2\t8 206 10\n",
      "26\t5767\t0.0\t2\t5478 259 29 1\n",
      "27\t200\t0.0\t2\t25 157 15 2 1\n",
      "28\t19\t0.0\t2\t0 3 15 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F07R15_TTCGTGATTCGATGACCTCTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F07R15_TTCGTGATTCGATGACCTCTAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F07R15_TTCGTGATTCGATGACCTCTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F07R15_TTCGTGATTCGATGACCTCTAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.42 s (67 µs/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,241 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                  14 (0.2%)\n",
      "Reads discarded as untrimmed:                9 (0.1%)\n",
      "Reads written (passing filters):         6,227 (99.6%)\n",
      "\n",
      "Total basepairs processed:     1,450,447 bp\n",
      "Total written (filtered):      1,128,547 bp (77.8%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6241 times; 3' trimmed: 6212 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t4\t0.0\t2\t0 0 3 1\n",
      "24\t64\t0.0\t2\t0 60 4\n",
      "25\t6150\t0.0\t2\t5529 553 68\n",
      "26\t21\t0.0\t2\t0 11 9 1\n",
      "27\t2\t0.0\t2\t0 0 1 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t7\t0.0\t2\t1 2 4\n",
      "25\t234\t0.0\t2\t18 206 8 2\n",
      "26\t5712\t0.0\t2\t5460 224 27 1\n",
      "27\t223\t0.0\t2\t22 184 17\n",
      "28\t35\t0.0\t2\t0 7 28\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F07R16_TTCGTGATTCGATGTCATCCAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F07R16_TTCGTGATTCGATGTCATCCAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F07R16_TTCGTGATTCGATGTCATCCAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F07R16_TTCGTGATTCGATGTCATCCAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.44 s (71 µs/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,245 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                  22 (0.4%)\n",
      "Reads discarded as untrimmed:                5 (0.1%)\n",
      "Reads written (passing filters):         6,223 (99.6%)\n",
      "\n",
      "Total basepairs processed:     1,452,145 bp\n",
      "Total written (filtered):      1,130,500 bp (77.9%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6245 times; 3' trimmed: 6182 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t4\t0.0\t2\t0 0 3 0 1\n",
      "24\t89\t0.0\t2\t0 67 20 2\n",
      "25\t6125\t0.0\t2\t5025 950 149 1\n",
      "26\t27\t0.0\t2\t0 17 9 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t14\t0.0\t2\t0 0 13 1\n",
      "25\t210\t0.0\t2\t6 193 11\n",
      "26\t5008\t0.0\t2\t4770 211 25 1 1\n",
      "27\t840\t0.0\t2\t186 614 39 1\n",
      "28\t106\t0.0\t2\t0 16 89 1\n",
      "29\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F07R17_TTCGTGATTCGATGACGAATAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F07R17_TTCGTGATTCGATGACGAATAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F07R17_TTCGTGATTCGATGACGAATAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F07R17_TTCGTGATTCGATGACGAATAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.41 s (65 µs/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,244 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads discarded as untrimmed:                6 (0.1%)\n",
      "Reads written (passing filters):         6,244 (99.9%)\n",
      "\n",
      "Total basepairs processed:     1,454,406 bp\n",
      "Total written (filtered):      1,134,959 bp (78.0%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6244 times; 3' trimmed: 6224 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t9\t0.0\t2\t0 0 9\n",
      "24\t71\t0.0\t2\t0 49 16 6\n",
      "25\t6145\t0.0\t2\t5432 642 71\n",
      "26\t16\t0.0\t2\t0 7 7 2\n",
      "27\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "24\t12\t0.0\t2\t0 1 10 1\n",
      "25\t194\t0.0\t2\t14 169 11\n",
      "26\t5628\t0.0\t2\t5389 191 48\n",
      "27\t360\t0.0\t2\t10 332 13 5\n",
      "28\t28\t0.0\t2\t0 4 22 2\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F07R18_TTCGTGATTCGATGCAATTGCCT_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F07R18_TTCGTGATTCGATGCAATTGCCT_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F07R18_TTCGTGATTCGATGCAATTGCCT_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F07R18_TTCGTGATTCGATGCAATTGCCT_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.42 s (68 µs/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,249 (100.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   2 (0.0%)\n",
      "Reads discarded as untrimmed:                1 (0.0%)\n",
      "Reads written (passing filters):         6,247 (100.0%)\n",
      "\n",
      "Total basepairs processed:     1,453,147 bp\n",
      "Total written (filtered):      1,134,117 bp (78.0%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6249 times; 3' trimmed: 6228 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t49\t0.0\t2\t0 39 10\n",
      "25\t6178\t0.0\t2\t5278 804 93 2 1\n",
      "26\t21\t0.0\t2\t0 11 8 2\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "21\t1\t0.0\t2\t0 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "24\t15\t0.0\t2\t1 2 12\n",
      "25\t213\t0.0\t2\t11 194 8\n",
      "26\t5442\t0.0\t2\t5236 171 35\n",
      "27\t522\t0.0\t2\t9 484 26 3\n",
      "28\t34\t0.0\t2\t0 2 30 1 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F07R19_TTCGTGATTCGATGTCCGACTAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F07R19_TTCGTGATTCGATGTCCGACTAA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F07R19_TTCGTGATTCGATGTCCGACTAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F07R19_TTCGTGATTCGATGTCCGACTAA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.45 s (72 µs/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,237 (99.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   1 (0.0%)\n",
      "Reads discarded as untrimmed:               13 (0.2%)\n",
      "Reads written (passing filters):         6,236 (99.8%)\n",
      "\n",
      "Total basepairs processed:     1,453,657 bp\n",
      "Total written (filtered):      1,134,156 bp (78.0%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6237 times; 3' trimmed: 6177 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t5\t0.0\t2\t0 0 4 1\n",
      "24\t46\t0.0\t2\t0 36 10\n",
      "25\t6164\t0.0\t2\t5461 599 103 1\n",
      "26\t20\t0.0\t2\t0 15 4 0 1\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "16\t1\t0.0\t1\t0 1\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "24\t27\t0.0\t2\t0 1 26\n",
      "25\t317\t0.0\t2\t17 288 12\n",
      "26\t5620\t0.0\t2\t5367 207 43 2 1\n",
      "27\t175\t0.0\t2\t18 143 14\n",
      "28\t33\t0.0\t2\t0 9 24\n",
      "29\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F07R20_TTCGTGATTCGATGATGGATCTG_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F07R20_TTCGTGATTCGATGATGGATCTG_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F07R20_TTCGTGATTCGATGATGGATCTG_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F07R20_TTCGTGATTCGATGATGGATCTG_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.46 s (74 µs/read; 0.82 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,248 (100.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                  27 (0.4%)\n",
      "Reads discarded as untrimmed:                2 (0.0%)\n",
      "Reads written (passing filters):         6,221 (99.5%)\n",
      "\n",
      "Total basepairs processed:     1,452,786 bp\n",
      "Total written (filtered):      1,130,192 bp (77.8%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6248 times; 3' trimmed: 6234 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t5\t0.0\t2\t0 0 4 1\n",
      "24\t60\t0.0\t2\t0 44 15 1\n",
      "25\t6158\t0.0\t2\t5446 622 88 2\n",
      "26\t21\t0.0\t2\t0 12 7 2\n",
      "27\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "21\t1\t0.0\t2\t0 0 1\n",
      "24\t21\t0.0\t2\t0 4 17\n",
      "25\t369\t0.0\t2\t25 334 6 4\n",
      "26\t5590\t0.0\t2\t5372 173 43 2\n",
      "27\t212\t0.0\t2\t36 164 12\n",
      "28\t40\t0.0\t2\t0 2 38\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F08R11_TTCCGATAACGATGATTCGAGGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F08R11_TTCCGATAACGATGATTCGAGGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F08R11_TTCCGATAACGATGATTCGAGGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F08R11_TTCCGATAACGATGATTCGAGGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F08R12_TTCCGATAACGATGAACCACCTA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F08R12_TTCCGATAACGATGAACCACCTA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F08R12_TTCCGATAACGATGAACCACCTA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F08R12_TTCCGATAACGATGAACCACCTA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.45 s (71 µs/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,246 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads discarded as untrimmed:                4 (0.1%)\n",
      "Reads written (passing filters):         6,246 (99.9%)\n",
      "\n",
      "Total basepairs processed:     1,454,727 bp\n",
      "Total written (filtered):      1,135,907 bp (78.1%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6246 times; 3' trimmed: 6220 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t4\t0.0\t2\t0 0 3 1\n",
      "24\t61\t0.0\t2\t0 49 9 3\n",
      "25\t6156\t0.0\t2\t5668 436 52\n",
      "26\t21\t0.0\t2\t0 16 4 1\n",
      "27\t4\t0.0\t2\t0 0 4\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "22\t1\t0.0\t2\t1\n",
      "23\t1\t0.0\t2\t1\n",
      "24\t11\t0.0\t2\t0 1 10\n",
      "25\t154\t0.0\t2\t4 144 5 1\n",
      "26\t5822\t0.0\t2\t5614 182 26\n",
      "27\t199\t0.0\t2\t23 166 9 1\n",
      "28\t31\t0.0\t2\t0 7 21 3\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F08R13_TTCCGATAACGATGTCCGTTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F08R13_TTCCGATAACGATGTCCGTTAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F08R13_TTCCGATAACGATGTCCGTTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F08R13_TTCCGATAACGATGTCCGTTAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.44 s (71 µs/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,249 (100.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads discarded as untrimmed:                1 (0.0%)\n",
      "Reads written (passing filters):         6,249 (100.0%)\n",
      "\n",
      "Total basepairs processed:     1,451,955 bp\n",
      "Total written (filtered):      1,133,481 bp (78.1%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6249 times; 3' trimmed: 6230 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t5\t0.0\t2\t0 0 5\n",
      "24\t56\t0.0\t2\t0 45 10 1\n",
      "25\t6154\t0.0\t2\t5644 451 59\n",
      "26\t32\t0.0\t2\t0 22 9 1\n",
      "27\t2\t0.0\t2\t0 0 1 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t1\t0.0\t2\t0 0 1\n",
      "24\t10\t0.0\t2\t0 0 9 1\n",
      "25\t169\t0.0\t2\t11 153 5\n",
      "26\t5814\t0.0\t2\t5591 193 30\n",
      "27\t212\t0.0\t2\t23 171 16 2\n",
      "28\t23\t0.0\t2\t0 1 22\n",
      "32\t1\t0.0\t2\t1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F08R14_TTCCGATAACGATGACACTCCAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F08R14_TTCCGATAACGATGACACTCCAA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F08R14_TTCCGATAACGATGACACTCCAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F08R14_TTCCGATAACGATGACACTCCAA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.42 s (67 µs/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,243 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   3 (0.0%)\n",
      "Reads discarded as untrimmed:                7 (0.1%)\n",
      "Reads written (passing filters):         6,240 (99.8%)\n",
      "\n",
      "Total basepairs processed:     1,450,821 bp\n",
      "Total written (filtered):      1,131,262 bp (78.0%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6243 times; 3' trimmed: 6213 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t3\t0.0\t2\t0 0 1 2\n",
      "24\t65\t0.0\t2\t0 57 7 1\n",
      "25\t6128\t0.0\t2\t5492 568 68\n",
      "26\t43\t0.0\t2\t0 34 6 3\n",
      "27\t4\t0.0\t2\t0 0 3 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "22\t1\t0.0\t2\t0 1\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "24\t16\t0.0\t2\t0 2 14\n",
      "25\t348\t0.0\t2\t23 311 12 2\n",
      "26\t5651\t0.0\t2\t5434 201 16\n",
      "27\t167\t0.0\t2\t5 144 15 3\n",
      "28\t26\t0.0\t2\t0 0 25 1\n",
      "29\t3\t0.0\t2\t0 0 2 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F08R15_TTCCGATAACGATGACCTCTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F08R15_TTCCGATAACGATGACCTCTAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F08R15_TTCCGATAACGATGACCTCTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F08R15_TTCCGATAACGATGACCTCTAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.41 s (65 µs/read; 0.92 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,244 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   4 (0.1%)\n",
      "Reads discarded as untrimmed:                6 (0.1%)\n",
      "Reads written (passing filters):         6,240 (99.8%)\n",
      "\n",
      "Total basepairs processed:     1,451,758 bp\n",
      "Total written (filtered):      1,132,033 bp (78.0%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6244 times; 3' trimmed: 6218 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "24\t62\t0.0\t2\t0 52 9 1\n",
      "25\t6156\t0.0\t2\t5519 559 78\n",
      "26\t17\t0.0\t2\t0 12 5\n",
      "27\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "13\t1\t0.0\t1\t0 1\n",
      "24\t17\t0.0\t2\t0 2 15\n",
      "25\t183\t0.0\t2\t7 169 7\n",
      "26\t5700\t0.0\t2\t5463 193 44\n",
      "27\t294\t0.0\t2\t23 255 16\n",
      "28\t21\t0.0\t2\t0 3 18\n",
      "29\t2\t0.0\t2\t0 0 1 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F08R16_TTCCGATAACGATGTCATCCAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F08R16_TTCCGATAACGATGTCATCCAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F08R16_TTCCGATAACGATGTCATCCAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F08R16_TTCCGATAACGATGTCATCCAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.43 s (69 µs/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,241 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   3 (0.0%)\n",
      "Reads discarded as untrimmed:                9 (0.1%)\n",
      "Reads written (passing filters):         6,238 (99.8%)\n",
      "\n",
      "Total basepairs processed:     1,450,451 bp\n",
      "Total written (filtered):      1,130,221 bp (77.9%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6241 times; 3' trimmed: 6223 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t78\t0.0\t2\t0 72 6\n",
      "25\t6138\t0.0\t2\t5739 355 44\n",
      "26\t20\t0.0\t2\t0 14 5 1\n",
      "27\t5\t0.0\t2\t0 0 5\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "15\t1\t0.0\t1\t1\n",
      "21\t1\t0.0\t2\t1\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t146\t0.0\t2\t4 137 4 1\n",
      "26\t5932\t0.0\t2\t5702 204 26\n",
      "27\t118\t0.0\t2\t13 99 6\n",
      "28\t12\t0.0\t2\t0 0 12\n",
      "29\t3\t0.0\t2\t0 1 2\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F08R17_TTCCGATAACGATGACGAATAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F08R17_TTCCGATAACGATGACGAATAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F08R17_TTCCGATAACGATGACGAATAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F08R17_TTCCGATAACGATGACGAATAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.45 s (72 µs/read; 0.84 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,246 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   2 (0.0%)\n",
      "Reads discarded as untrimmed:                4 (0.1%)\n",
      "Reads written (passing filters):         6,244 (99.9%)\n",
      "\n",
      "Total basepairs processed:     1,452,687 bp\n",
      "Total written (filtered):      1,134,236 bp (78.1%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6246 times; 3' trimmed: 6201 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t3\t0.0\t2\t0 0 3\n",
      "24\t84\t0.0\t2\t0 67 13 4\n",
      "25\t6130\t0.0\t2\t5612 452 65 1\n",
      "26\t28\t0.0\t2\t0 19 9\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t2\t0.0\t2\t0 1 1\n",
      "24\t19\t0.0\t2\t0 3 14 2\n",
      "25\t205\t0.0\t2\t11 186 8\n",
      "26\t5814\t0.0\t2\t5549 226 38 1\n",
      "27\t138\t0.0\t2\t7 122 9\n",
      "28\t23\t0.0\t2\t0 0 21 2\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F08R18_TTCCGATAACGATGCAATTGCCT_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F08R18_TTCCGATAACGATGCAATTGCCT_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F08R18_TTCCGATAACGATGCAATTGCCT_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F08R18_TTCCGATAACGATGCAATTGCCT_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.41 s (66 µs/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,246 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   2 (0.0%)\n",
      "Reads discarded as untrimmed:                4 (0.1%)\n",
      "Reads written (passing filters):         6,244 (99.9%)\n",
      "\n",
      "Total basepairs processed:     1,457,578 bp\n",
      "Total written (filtered):      1,138,459 bp (78.1%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6246 times; 3' trimmed: 6218 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "24\t69\t0.0\t2\t0 58 11\n",
      "25\t6157\t0.0\t2\t5358 691 106 2\n",
      "26\t17\t0.0\t2\t0 9 7 1\n",
      "27\t1\t0.0\t2\t0 0 0 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "24\t20\t0.0\t2\t0 4 16\n",
      "25\t315\t0.0\t2\t9 289 14 3\n",
      "26\t5171\t0.0\t2\t4935 200 35 1\n",
      "27\t653\t0.0\t2\t386 252 15\n",
      "28\t55\t0.0\t2\t1 12 42\n",
      "29\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F08R19_TTCCGATAACGATGTCCGACTAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F08R19_TTCCGATAACGATGTCCGACTAA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F08R19_TTCCGATAACGATGTCCGACTAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F08R19_TTCCGATAACGATGTCCGACTAA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.40 s (64 µs/read; 0.94 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,243 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   4 (0.1%)\n",
      "Reads discarded as untrimmed:                7 (0.1%)\n",
      "Reads written (passing filters):         6,239 (99.8%)\n",
      "\n",
      "Total basepairs processed:     1,456,897 bp\n",
      "Total written (filtered):      1,137,352 bp (78.1%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6243 times; 3' trimmed: 6191 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t5\t0.0\t2\t0 0 4 1\n",
      "24\t84\t0.0\t2\t0 71 12 1\n",
      "25\t6125\t0.0\t2\t5232 787 104 2\n",
      "26\t26\t0.0\t2\t0 18 7 1\n",
      "27\t3\t0.0\t2\t0 0 1 2\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t1\t0.0\t2\t0 0 0 1\n",
      "24\t16\t0.0\t2\t0 2 14\n",
      "25\t311\t0.0\t2\t9 292 8 2\n",
      "26\t5270\t0.0\t2\t5015 214 40 1\n",
      "27\t537\t0.0\t2\t158 359 17 3\n",
      "28\t54\t0.0\t2\t0 8 46\n",
      "29\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F08R20_TTCCGATAACGATGATGGATCTG_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F08R20_TTCCGATAACGATGATGGATCTG_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F08R20_TTCCGATAACGATGATGGATCTG_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F08R20_TTCCGATAACGATGATGGATCTG_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.41 s (66 µs/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,239 (99.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   4 (0.1%)\n",
      "Reads discarded as untrimmed:               11 (0.2%)\n",
      "Reads written (passing filters):         6,235 (99.8%)\n",
      "\n",
      "Total basepairs processed:     1,454,388 bp\n",
      "Total written (filtered):      1,133,998 bp (78.0%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6239 times; 3' trimmed: 6212 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t8\t0.0\t2\t0 0 7 1\n",
      "24\t103\t0.0\t2\t0 80 19 4\n",
      "25\t6101\t0.0\t2\t5353 661 86 1\n",
      "26\t24\t0.0\t2\t0 14 9 0 1\n",
      "27\t3\t0.0\t2\t0 0 2 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "13\t1\t0.0\t1\t0 1\n",
      "23\t3\t0.0\t2\t0 1 2\n",
      "24\t28\t0.0\t2\t1 6 19 1 1\n",
      "25\t389\t0.0\t2\t24 344 18 3\n",
      "26\t5500\t0.0\t2\t5248 212 38 2\n",
      "27\t253\t0.0\t2\t53 180 19 1\n",
      "28\t37\t0.0\t2\t0 11 26\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F09R11_TGAGCGGAACGATGATTCGAGGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F09R11_TGAGCGGAACGATGATTCGAGGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F09R11_TGAGCGGAACGATGATTCGAGGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F09R11_TGAGCGGAACGATGATTCGAGGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F09R12_TGAGCGGAACGATGAACCACCTA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F09R12_TGAGCGGAACGATGAACCACCTA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F09R12_TGAGCGGAACGATGAACCACCTA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F09R12_TGAGCGGAACGATGAACCACCTA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.42 s (67 µs/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,245 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads discarded as untrimmed:                5 (0.1%)\n",
      "Reads written (passing filters):         6,245 (99.9%)\n",
      "\n",
      "Total basepairs processed:     1,450,417 bp\n",
      "Total written (filtered):      1,131,236 bp (78.0%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6245 times; 3' trimmed: 6238 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t2\t0.0\t2\t0 0 1 1\n",
      "24\t49\t0.0\t2\t0 42 5 2\n",
      "25\t6176\t0.0\t2\t5747 396 33\n",
      "26\t15\t0.0\t2\t0 10 5\n",
      "27\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "14\t1\t0.0\t1\t0 1\n",
      "20\t1\t0.0\t2\t1\n",
      "24\t17\t0.0\t2\t0 1 16\n",
      "25\t269\t0.0\t2\t2 263 3 1\n",
      "26\t5876\t0.0\t2\t5728 128 19 1\n",
      "27\t67\t0.0\t2\t9 55 2 1\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F09R13_TGAGCGGAACGATGTCCGTTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F09R13_TGAGCGGAACGATGTCCGTTAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F09R13_TGAGCGGAACGATGTCCGTTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F09R13_TGAGCGGAACGATGTCCGTTAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.42 s (67 µs/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,241 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                  63 (1.0%)\n",
      "Reads discarded as untrimmed:                9 (0.1%)\n",
      "Reads written (passing filters):         6,178 (98.8%)\n",
      "\n",
      "Total basepairs processed:     1,443,949 bp\n",
      "Total written (filtered):      1,122,098 bp (77.7%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6241 times; 3' trimmed: 6215 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t5\t0.0\t2\t0 0 5\n",
      "24\t52\t0.0\t2\t0 45 6 1\n",
      "25\t6161\t0.0\t2\t5551 523 86 1\n",
      "26\t22\t0.0\t2\t0 12 9 1\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1\t0.0\t1\t0 1\n",
      "24\t10\t0.0\t2\t0 0 10\n",
      "25\t201\t0.0\t2\t8 183 9 1\n",
      "26\t5610\t0.0\t2\t5376 201 32 1\n",
      "27\t343\t0.0\t2\t142 187 13 1\n",
      "28\t46\t0.0\t2\t0 8 38\n",
      "29\t3\t0.0\t2\t0 0 3\n",
      "30\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F09R14_TGAGCGGAACGATGACACTCCAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F09R14_TGAGCGGAACGATGACACTCCAA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F09R14_TGAGCGGAACGATGACACTCCAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F09R14_TGAGCGGAACGATGACACTCCAA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.42 s (67 µs/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,244 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                  28 (0.4%)\n",
      "Reads discarded as untrimmed:                6 (0.1%)\n",
      "Reads written (passing filters):         6,216 (99.5%)\n",
      "\n",
      "Total basepairs processed:     1,452,554 bp\n",
      "Total written (filtered):      1,130,222 bp (77.8%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6244 times; 3' trimmed: 6219 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t6\t0.0\t2\t0 0 6\n",
      "24\t39\t0.0\t2\t0 37 2\n",
      "25\t6171\t0.0\t2\t5398 683 90\n",
      "26\t25\t0.0\t2\t0 22 2 1\n",
      "27\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "18\t1\t0.0\t1\t0 1\n",
      "20\t1\t0.0\t2\t1\n",
      "23\t2\t0.0\t2\t0 1 1\n",
      "24\t15\t0.0\t2\t0 1 14\n",
      "25\t272\t0.0\t2\t21 242 9\n",
      "26\t5427\t0.0\t2\t5213 192 22\n",
      "27\t443\t0.0\t2\t139 286 18\n",
      "28\t53\t0.0\t2\t0 18 35\n",
      "29\t5\t0.0\t2\t0 0 4 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F09R15_TGAGCGGAACGATGACCTCTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F09R15_TGAGCGGAACGATGACCTCTAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F09R15_TGAGCGGAACGATGACCTCTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F09R15_TGAGCGGAACGATGACCTCTAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.42 s (68 µs/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,241 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                  18 (0.3%)\n",
      "Reads discarded as untrimmed:                9 (0.1%)\n",
      "Reads written (passing filters):         6,223 (99.6%)\n",
      "\n",
      "Total basepairs processed:     1,451,866 bp\n",
      "Total written (filtered):      1,129,119 bp (77.8%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6241 times; 3' trimmed: 6223 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t51\t0.0\t2\t0 44 6 1\n",
      "25\t6170\t0.0\t2\t5464 636 69 1\n",
      "26\t19\t0.0\t2\t0 16 3\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t17\t0.0\t2\t0 2 15\n",
      "25\t262\t0.0\t2\t15 241 6\n",
      "26\t5573\t0.0\t2\t5372 180 20 1\n",
      "27\t333\t0.0\t2\t59 261 13\n",
      "28\t37\t0.0\t2\t0 12 24 1\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F09R16_TGAGCGGAACGATGTCATCCAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F09R16_TGAGCGGAACGATGTCATCCAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F09R16_TGAGCGGAACGATGTCATCCAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F09R16_TGAGCGGAACGATGTCATCCAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.43 s (69 µs/read; 0.87 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,249 (100.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                  68 (1.1%)\n",
      "Reads discarded as untrimmed:                1 (0.0%)\n",
      "Reads written (passing filters):         6,181 (98.9%)\n",
      "\n",
      "Total basepairs processed:     1,447,619 bp\n",
      "Total written (filtered):      1,119,460 bp (77.3%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6249 times; 3' trimmed: 6205 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "24\t46\t0.0\t2\t0 39 7\n",
      "25\t6180\t0.0\t2\t5633 496 51\n",
      "26\t20\t0.0\t2\t0 18 2\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t5\t0.0\t2\t0 1 4\n",
      "24\t13\t0.0\t2\t0 0 13\n",
      "25\t256\t0.0\t2\t7 243 6\n",
      "26\t5758\t0.0\t2\t5579 165 14\n",
      "27\t156\t0.0\t2\t3 144 9\n",
      "28\t17\t0.0\t2\t0 0 17\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F09R17_TGAGCGGAACGATGACGAATAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F09R17_TGAGCGGAACGATGACGAATAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F09R17_TGAGCGGAACGATGACGAATAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F09R17_TGAGCGGAACGATGACGAATAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.41 s (66 µs/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,246 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                  43 (0.7%)\n",
      "Reads discarded as untrimmed:                4 (0.1%)\n",
      "Reads written (passing filters):         6,203 (99.2%)\n",
      "\n",
      "Total basepairs processed:     1,450,794 bp\n",
      "Total written (filtered):      1,125,733 bp (77.6%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6246 times; 3' trimmed: 6230 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "24\t58\t0.0\t2\t0 50 7 1\n",
      "25\t6168\t0.0\t2\t5519 592 57\n",
      "26\t16\t0.0\t2\t0 10 6\n",
      "27\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t2\t0.0\t2\t0 0 2\n",
      "24\t13\t0.0\t2\t0 1 12\n",
      "25\t267\t0.0\t2\t26 229 12\n",
      "26\t5669\t0.0\t2\t5441 203 25\n",
      "27\t257\t0.0\t2\t36 215 6\n",
      "28\t22\t0.0\t2\t0 4 17 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F09R18_TGAGCGGAACGATGCAATTGCCT_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F09R18_TGAGCGGAACGATGCAATTGCCT_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F09R18_TGAGCGGAACGATGCAATTGCCT_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F09R18_TGAGCGGAACGATGCAATTGCCT_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.42 s (68 µs/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,248 (100.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads discarded as untrimmed:                2 (0.0%)\n",
      "Reads written (passing filters):         6,248 (100.0%)\n",
      "\n",
      "Total basepairs processed:     1,451,084 bp\n",
      "Total written (filtered):      1,133,013 bp (78.1%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6248 times; 3' trimmed: 6209 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t3\t0.0\t2\t0 0 3\n",
      "24\t52\t0.0\t2\t0 45 7\n",
      "25\t6175\t0.0\t2\t5456 660 59\n",
      "26\t16\t0.0\t2\t0 11 4 1\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t13\t0.0\t2\t0 1 12\n",
      "25\t312\t0.0\t2\t8 298 6\n",
      "26\t5564\t0.0\t2\t5382 154 28\n",
      "27\t297\t0.0\t2\t30 259 7 1\n",
      "28\t22\t0.0\t2\t0 1 21\n",
      "29\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F09R19_TGAGCGGAACGATGTCCGACTAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F09R19_TGAGCGGAACGATGTCCGACTAA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F09R19_TGAGCGGAACGATGTCCGACTAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F09R19_TGAGCGGAACGATGTCCGACTAA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.37 s (92413 µs/read; 0.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                       4\n",
      "Reads with adapters:                         4 (100.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads discarded as untrimmed:                0 (0.0%)\n",
      "Reads written (passing filters):             4 (100.0%)\n",
      "\n",
      "Total basepairs processed:           930 bp\n",
      "Total written (filtered):            726 bp (78.1%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 4 times; 3' trimmed: 4 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "25\t4\t0.0\t2\t3 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "26\t4\t0.0\t2\t3 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F09R20_TGAGCGGAACGATGATGGATCTG_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F09R20_TGAGCGGAACGATGATGGATCTG_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F09R20_TGAGCGGAACGATGATGGATCTG_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F09R20_TGAGCGGAACGATGATGGATCTG_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.42 s (67 µs/read; 0.90 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,241 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   1 (0.0%)\n",
      "Reads discarded as untrimmed:                9 (0.1%)\n",
      "Reads written (passing filters):         6,240 (99.8%)\n",
      "\n",
      "Total basepairs processed:     1,448,819 bp\n",
      "Total written (filtered):      1,129,044 bp (77.9%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6241 times; 3' trimmed: 6220 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t9\t0.0\t2\t0 0 8 1\n",
      "24\t85\t0.0\t2\t0 72 11 2\n",
      "25\t6118\t0.0\t2\t5332 684 100 2\n",
      "26\t20\t0.0\t2\t0 15 5\n",
      "27\t9\t0.0\t2\t0 0 9\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "20\t2\t0.0\t2\t0 0 2\n",
      "23\t2\t0.0\t2\t0 0 1 1\n",
      "24\t26\t0.0\t2\t0 10 16\n",
      "25\t310\t0.0\t2\t50 246 12 2\n",
      "26\t5523\t0.0\t2\t5239 240 44\n",
      "27\t311\t0.0\t2\t23 270 18\n",
      "28\t44\t0.0\t2\t0 4 38 2\n",
      "29\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F10R11_CTGACCGAACGATGATTCGAGGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F10R11_CTGACCGAACGATGATTCGAGGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F10R11_CTGACCGAACGATGATTCGAGGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F10R11_CTGACCGAACGATGATTCGAGGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "No reads processed!\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F10R12_CTGACCGAACGATGAACCACCTA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F10R12_CTGACCGAACGATGAACCACCTA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F10R12_CTGACCGAACGATGAACCACCTA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F10R12_CTGACCGAACGATGAACCACCTA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.37 s (4688 µs/read; 0.01 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                      78\n",
      "Reads with adapters:                        78 (100.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads discarded as untrimmed:                0 (0.0%)\n",
      "Reads written (passing filters):            78 (100.0%)\n",
      "\n",
      "Total basepairs processed:        18,181 bp\n",
      "Total written (filtered):         14,230 bp (78.3%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 78 times; 3' trimmed: 77 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "25\t78\t0.0\t2\t67 10 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t1\t0.0\t2\t0 0 1\n",
      "25\t2\t0.0\t2\t0 2\n",
      "26\t71\t0.0\t2\t65 6\n",
      "27\t3\t0.0\t2\t1 2\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F10R13_CTGACCGAACGATGTCCGTTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F10R13_CTGACCGAACGATGTCCGTTAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F10R13_CTGACCGAACGATGTCCGTTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F10R13_CTGACCGAACGATGTCCGTTAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.43 s (69 µs/read; 0.88 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,245 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                  12 (0.2%)\n",
      "Reads discarded as untrimmed:                5 (0.1%)\n",
      "Reads written (passing filters):         6,233 (99.7%)\n",
      "\n",
      "Total basepairs processed:     1,457,433 bp\n",
      "Total written (filtered):      1,138,344 bp (78.1%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6245 times; 3' trimmed: 6218 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t6\t0.0\t2\t0 0 4 2\n",
      "24\t87\t0.0\t2\t0 70 14 3\n",
      "25\t6133\t0.0\t2\t5486 597 50\n",
      "26\t16\t0.0\t2\t0 12 2 1 1\n",
      "27\t3\t0.0\t2\t0 0 3\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "22\t1\t0.0\t2\t0 1\n",
      "24\t13\t0.0\t2\t0 2 10 0 1\n",
      "25\t326\t0.0\t2\t5 308 12 1\n",
      "26\t5701\t0.0\t2\t5453 210 37 1\n",
      "27\t170\t0.0\t2\t2 157 9 2\n",
      "28\t7\t0.0\t2\t0 0 5 2\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F10R14_CTGACCGAACGATGACACTCCAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F10R14_CTGACCGAACGATGACACTCCAA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F10R14_CTGACCGAACGATGACACTCCAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F10R14_CTGACCGAACGATGACACTCCAA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.38 s (61 µs/read; 0.99 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,248 (100.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:               6,215 (99.4%)\n",
      "Reads discarded as untrimmed:                0 (0.0%)\n",
      "Reads written (passing filters):            35 (0.6%)\n",
      "\n",
      "Total basepairs processed:       330,206 bp\n",
      "Total written (filtered):          6,372 bp (1.9%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6248 times; 3' trimmed: 5002 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t8\t0.0\t2\t0 0 1 3 4\n",
      "24\t76\t0.0\t2\t0 20 16 39 1\n",
      "25\t6140\t0.0\t2\t1971 1368 2777 23 1\n",
      "26\t22\t0.0\t2\t0 6 6 9 1\n",
      "27\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "15\t1\t0.0\t1\t0 1\n",
      "17\t1\t0.0\t1\t0 0 1\n",
      "23\t4\t0.0\t2\t0 0 4\n",
      "24\t2624\t0.0\t2\t0 3 2549 66 6\n",
      "25\t1134\t0.0\t2\t2 1092 36 4\n",
      "26\t924\t0.0\t2\t758 28 135 3\n",
      "27\t243\t0.0\t2\t2 234 7\n",
      "28\t71\t0.0\t2\t0 1 68 1 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F10R15_CTGACCGAACGATGACCTCTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F10R15_CTGACCGAACGATGACCTCTAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F10R15_CTGACCGAACGATGACCTCTAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F10R15_CTGACCGAACGATGACCTCTAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.42 s (1041 µs/read; 0.06 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                     404\n",
      "Reads with adapters:                       404 (100.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                 275 (68.1%)\n",
      "Reads discarded as untrimmed:                0 (0.0%)\n",
      "Reads written (passing filters):           129 (31.9%)\n",
      "\n",
      "Total basepairs processed:        44,470 bp\n",
      "Total written (filtered):         23,458 bp (52.8%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 404 times; 3' trimmed: 377 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t4\t0.0\t2\t0 3 1\n",
      "25\t395\t0.0\t2\t242 45 107 1\n",
      "26\t2\t0.0\t2\t0 2\n",
      "27\t3\t0.0\t2\t0 0 2 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "24\t98\t0.0\t2\t0 0 97 1\n",
      "25\t13\t0.0\t2\t0 13\n",
      "26\t226\t0.0\t2\t215 7 4\n",
      "27\t32\t0.0\t2\t0 29 2 1\n",
      "28\t7\t0.0\t2\t0 1 6\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F10R16_CTGACCGAACGATGTCATCCAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F10R16_CTGACCGAACGATGTCATCCAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F10R16_CTGACCGAACGATGTCATCCAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F10R16_CTGACCGAACGATGTCATCCAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.44 s (71 µs/read; 0.85 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,247 (100.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:               3,626 (58.0%)\n",
      "Reads discarded as untrimmed:                3 (0.0%)\n",
      "Reads written (passing filters):         2,621 (41.9%)\n",
      "\n",
      "Total basepairs processed:       818,835 bp\n",
      "Total written (filtered):        480,637 bp (58.7%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6247 times; 3' trimmed: 5969 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t6\t0.0\t2\t0 0 2 2 2\n",
      "24\t91\t0.0\t2\t0 69 13 9\n",
      "25\t6125\t0.0\t2\t4232 1066 822 5\n",
      "26\t19\t0.0\t2\t0 8 6 4 1\n",
      "27\t6\t0.0\t2\t0 0 5 0 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "24\t489\t0.0\t2\t0 0 475 12 2\n",
      "25\t245\t0.0\t2\t4 230 10 1\n",
      "26\t4210\t0.0\t2\t3956 158 92 3 1\n",
      "27\t777\t0.0\t2\t2 745 28 2\n",
      "28\t246\t0.0\t2\t0 3 240 2 1\n",
      "29\t2\t0.0\t2\t0 0 2\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F10R17_CTGACCGAACGATGACGAATAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F10R17_CTGACCGAACGATGACGAATAGA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F10R17_CTGACCGAACGATGACGAATAGA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F10R17_CTGACCGAACGATGACGAATAGA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.42 s (68 µs/read; 0.89 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,245 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads discarded as untrimmed:                5 (0.1%)\n",
      "Reads written (passing filters):         6,245 (99.9%)\n",
      "\n",
      "Total basepairs processed:     1,460,385 bp\n",
      "Total written (filtered):      1,141,150 bp (78.1%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6245 times; 3' trimmed: 6238 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t4\t0.0\t2\t0 0 3 1\n",
      "24\t85\t0.0\t2\t0 66 18 1\n",
      "25\t6132\t0.0\t2\t5376 695 61\n",
      "26\t21\t0.0\t2\t0 17 4\n",
      "27\t3\t0.0\t2\t0 0 2 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "15\t1\t0.0\t1\t1\n",
      "24\t24\t0.0\t2\t0 3 21\n",
      "25\t387\t0.0\t2\t10 359 18\n",
      "26\t5564\t0.0\t2\t5305 230 28 1\n",
      "27\t251\t0.0\t2\t54 181 14 2\n",
      "28\t11\t0.0\t2\t0 4 7\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F10R18_CTGACCGAACGATGCAATTGCCT_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F10R18_CTGACCGAACGATGCAATTGCCT_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F10R18_CTGACCGAACGATGCAATTGCCT_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F10R18_CTGACCGAACGATGCAATTGCCT_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.39 s (62 µs/read; 0.96 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,240 (99.8%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:               6,242 (99.9%)\n",
      "Reads discarded as untrimmed:                0 (0.0%)\n",
      "Reads written (passing filters):             8 (0.1%)\n",
      "\n",
      "Total basepairs processed:       308,997 bp\n",
      "Total written (filtered):          1,453 bp (0.5%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6240 times; 3' trimmed: 4804 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t1\t0.0\t2\t0 0 1\n",
      "24\t39\t0.0\t2\t0 9 0 29 1\n",
      "25\t6136\t0.0\t2\t1455 28 4620 31 2\n",
      "26\t47\t0.0\t2\t0 13 1 30 3\n",
      "27\t17\t0.0\t2\t0 0 7 0 10\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t5\t0.0\t2\t0 0 5\n",
      "24\t4705\t0.0\t2\t0 0 4601 89 15\n",
      "25\t17\t0.0\t2\t0 5 11 1\n",
      "26\t70\t0.0\t2\t67 3\n",
      "27\t4\t0.0\t2\t0 4\n",
      "28\t3\t0.0\t2\t0 0 2 0 1\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F10R19_CTGACCGAACGATGTCCGACTAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F10R19_CTGACCGAACGATGTCCGACTAA_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F10R19_CTGACCGAACGATGTCCGACTAA_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F10R19_CTGACCGAACGATGTCCGACTAA_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n",
      "Finished in 0.41 s (66 µs/read; 0.91 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                   6,250\n",
      "Reads with adapters:                     6,245 (99.9%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads discarded as untrimmed:                5 (0.1%)\n",
      "Reads written (passing filters):         6,245 (99.9%)\n",
      "\n",
      "Total basepairs processed:     1,486,455 bp\n",
      "Total written (filtered):      1,167,805 bp (78.6%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 6245 times; 3' trimmed: 6234 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "23\t5\t0.0\t2\t0 0 4 1\n",
      "24\t56\t0.0\t2\t0 47 7 2\n",
      "25\t6165\t0.0\t2\t5220 895 50\n",
      "26\t18\t0.0\t2\t0 13 5\n",
      "27\t1\t0.0\t2\t0 0 1\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "17\t1\t0.0\t1\t0 0 1\n",
      "24\t28\t0.0\t2\t0 9 18 1\n",
      "25\t736\t0.0\t2\t15 702 19\n",
      "26\t5358\t0.0\t2\t5191 147 20\n",
      "27\t104\t0.0\t2\t3 97 2 2\n",
      "28\t7\t0.0\t2\t0 0 7\n",
      "\n",
      "Command: cutadapt --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F10R20_CTGACCGAACGATGATGGATCTG_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F10R20_CTGACCGAACGATGATGGATCTG_L001_R1_001.fastq.gz\n",
      "\n",
      "This is cutadapt 3.5 with Python 3.8.12\n",
      "Command line parameters: --cores 16 --error-rate 0.1 --times 1 --overlap 3 --minimum-length 170 -o /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/q2-CasavaOneEightSingleLanePerSampleDirFmt-flgo81t7/F10R20_CTGACCGAACGATGATGGATCTG_L001_R1_001.fastq.gz --adapter ^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC --discard-untrimmed /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-zgkuymgn/db4f0261-44f7-473c-ba38-c72c16af0e2a/data/F10R20_CTGACCGAACGATGATGGATCTG_L001_R1_001.fastq.gz\n",
      "Processing reads on 16 cores in single-end mode ...\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Finished in 0.37 s (123871 µs/read; 0.00 M reads/minute).\n",
      "\n",
      "=== Summary ===\n",
      "\n",
      "Total reads processed:                       3\n",
      "Reads with adapters:                         3 (100.0%)\n",
      "\n",
      "== Read fate breakdown ==\n",
      "Reads that were too short:                   0 (0.0%)\n",
      "Reads discarded as untrimmed:                0 (0.0%)\n",
      "Reads written (passing filters):             3 (100.0%)\n",
      "\n",
      "Total basepairs processed:           698 bp\n",
      "Total written (filtered):            545 bp (78.1%)\n",
      "\n",
      "=== Adapter 3 ===\n",
      "\n",
      "Sequence: GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC; Type: linked; Length: 25+26; 5' trimmed: 3 times; 3' trimmed: 3 times\n",
      "\n",
      "Minimum overlap: 25+3\n",
      "No. of allowed errors: 2\n",
      "\n",
      "No. of allowed errors:\n",
      "1-9 bp: 0; 10-19 bp: 1; 20-26 bp: 2\n",
      "\n",
      "Overview of removed sequences at 5' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "25\t3\t0.0\t2\t3\n",
      "\n",
      "\n",
      "\n",
      "Overview of removed sequences at 3' end\n",
      "length\tcount\texpect\tmax.err\terror counts\n",
      "26\t3\t0.0\t2\t3\n"
     ]
    }
   ],
   "source": [
    "#Trimming ANML primers off the demultiplexed reads\n",
    "ANML_primers = [\"^GGTCAACAAATCATAAAGATATTGG...GGATTTGGAAATTGATTAGTWCCATC\"]\n",
    "\n",
    "trimmed_demux = cutadapt.methods.trim_single(demux, \n",
    "                                             cores=16, \n",
    "                                             adapter = ANML_primers, \n",
    "                                             indels = True,\n",
    "                                             minimum_length = 170, \n",
    "                                             discard_untrimmed = True)\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "1750582b",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-i69cw43d')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: c61f6e7c-1df1-4b4b-b5e0-cbfcf9420f18>"
      ]
     },
     "execution_count": 4,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "summarize(trimmed_demux.trimmed_sequences).visualization"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "4892883e",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Running external command line application(s). This may print messages to stdout and/or stderr.\n",
      "The command(s) being run are below. These commands cannot be manually re-run as they will depend on temporary files that no longer exist.\n",
      "\n",
      "Command: run_dada_single.R /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-3yyuz436/e64260d4-c583-4db5-bce3-ed858c0daca0/data /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/tmpc_oxa8rx/output.tsv.biom /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/tmpc_oxa8rx/track.tsv /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/tmpc_oxa8rx 0 0 2.0 2 Inf independent consensus 1.0 0 1000000 NULL 16\n",
      "\n",
      "R version 4.0.5 (2021-03-31) \n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "Loading required package: Rcpp\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "DADA2: 1.18.0 / Rcpp: 1.0.7 / RcppParallel: 5.1.4 \n",
      "1) Filtering "
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "The filter removed all reads: /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/tmpc_oxa8rx/F01R18_CTAAGGTAACGATGCAATTGCCT_L001_R1_001.fastq.gz not written.\n",
      "The filter removed all reads: /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/tmpc_oxa8rx/F01R14_CTAAGGTAACGATGACACTCCAA_L001_R1_001.fastq.gz not written.\n",
      "The filter removed all reads: /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/tmpc_oxa8rx/F01R16_CTAAGGTAACGATGTCATCCAGA_L001_R1_001.fastq.gz not written.\n",
      "The filter removed all reads: /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/tmpc_oxa8rx/F01R13_CTAAGGTAACGATGTCCGTTAGA_L001_R1_001.fastq.gz not written.\n",
      "The filter removed all reads: /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/tmpc_oxa8rx/F02R11_TAAGGAGAACGATGATTCGAGGA_L001_R1_001.fastq.gz not written.\n",
      "The filter removed all reads: /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/tmpc_oxa8rx/F01R17_CTAAGGTAACGATGACGAATAGA_L001_R1_001.fastq.gz not written.\n",
      "The filter removed all reads: /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/tmpc_oxa8rx/F01R20_CTAAGGTAACGATGATGGATCTG_L001_R1_001.fastq.gz not written.\n",
      "The filter removed all reads: /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/tmpc_oxa8rx/F01R19_CTAAGGTAACGATGTCCGACTAA_L001_R1_001.fastq.gz not written.\n",
      "The filter removed all reads: /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/tmpc_oxa8rx/F04R11_TCGCAATTACGATGATTCGAGGA_L001_R1_001.fastq.gz not written.\n",
      "The filter removed all reads: /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/tmpc_oxa8rx/F05R11_CAGAAGGAACGATGATTCGAGGA_L001_R1_001.fastq.gz not written.\n",
      "The filter removed all reads: /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/tmpc_oxa8rx/F06R11_CTGCAAGTTCGATGATTCGAGGA_L001_R1_001.fastq.gz not written.\n",
      "The filter removed all reads: /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/tmpc_oxa8rx/F07R11_TTCGTGATTCGATGATTCGAGGA_L001_R1_001.fastq.gz not written.\n",
      "The filter removed all reads: /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/tmpc_oxa8rx/F08R11_TTCCGATAACGATGATTCGAGGA_L001_R1_001.fastq.gz not written.\n",
      "The filter removed all reads: /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/tmpc_oxa8rx/F09R11_TGAGCGGAACGATGATTCGAGGA_L001_R1_001.fastq.gz not written.\n",
      "The filter removed all reads: /var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/tmpc_oxa8rx/F10R11_CTGACCGAACGATGATTCGAGGA_L001_R1_001.fastq.gz not written.\n",
      "Some input samples had no reads pass the filter.\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "..xx.xxxxxx...................x.........x.........x.........x.........x.........x.........x.........\n",
      "2) Learning Error Rates\n",
      "60837974 total bases in 377153 reads from 85 samples will be used for learning the error rates.\n",
      "3) Denoise samples .....................................................................................\n",
      "4) Remove chimeras (method = consensus)\n",
      "5) Report read numbers through the pipeline\n",
      "6) Write output\n"
     ]
    }
   ],
   "source": [
    "#DADA2\n",
    "feature_table, rep_seqs, stats = denoise_single(trimmed_demux.trimmed_sequences,\n",
    "                                                trunc_len = 0, \n",
    "                                                n_threads = 0)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "b50bd6c8",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div><img onload=\"(function(div, url){\n",
       "if (typeof require !== 'undefined') {\n",
       "    var baseURL = require.toUrl('').split('/').slice(0, -2).join('/');\n",
       "} else {\n",
       "    var baseURL = JSON.parse(\n",
       "        document.getElementById('jupyter-config-data').innerHTML\n",
       "    ).baseUrl.slice(0, -1);\n",
       "}\n",
       "url = baseURL + url;\n",
       "fetch(url).then(function(res) {\n",
       "    if (res.status === 404) {\n",
       "        div.innerHTML = 'Install QIIME 2 Jupyter extension with:<br />' +\n",
       "                        '<code>jupyter serverextension enable --py qiime2' +\n",
       "                        ' --sys-prefix</code><br />then restart your server.' +\n",
       "                        '<br /><br />(Interactive output not available on ' +\n",
       "                        'static notebook viewer services like nbviewer.)';\n",
       "    } else if (res.status === 409) {\n",
       "        div.innerHTML = 'Visualization no longer in scope. Re-run this cell' +\n",
       "                        ' to see the visualization.';\n",
       "    } else if (res.ok) {\n",
       "        url = res.url;\n",
       "        div.innerHTML = '<iframe src=\\'' + url + '\\' style=\\'' +\n",
       "                        'width: 100%; height: 700px; border: 0;\\'>' +\n",
       "                        '</iframe><hr />Open in a: <a href=\\'' + url + '\\'' +\n",
       "                        ' target=\\'_blank\\'>new window</a>'\n",
       "    } else {\n",
       "        div.innerHTML = 'Something has gone wrong. Check notebook server for' +\n",
       "                        ' errors.';\n",
       "    }\n",
       "});\n",
       "})(this.parentElement, '/qiime2/redirect?location=/var/folders/hh/1l3l9sld7h5dffby9kzw1ptc0000gn/T/qiime2-archive-0qooofpi')\" src=\"data:image/gif;base64,R0lGODlhAQABAIAAAP///wAAACH5BAEAAAAALAAAAAABAAEAAAICRAEAOw==\" /></div>"
      ],
      "text/plain": [
       "<visualization: Visualization uuid: e0d05147-ed27-4bab-98a8-0ce43f87c9fb>"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "feature_table.save('../PerSampleFiltering/feature_table.qza')\n",
    "summarize_table(feature_table).visualization"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "8a6d9055",
   "metadata": {},
   "outputs": [],
   "source": [
    "# Extract .biom from feature table file if necessary\n",
    "def extract_all(file, dest):\n",
    "    with tempfile.TemporaryDirectory() as temp:\n",
    "        file.export_data(temp)\n",
    "        temp_pathlib = pathlib.Path(temp)\n",
    "        for file in temp_pathlib.iterdir():\n",
    "            shutil.copy(file, dest)\n",
    "extract_all(feature_table, 'out')"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
